--- a/third_party/webrtc/modules/audio_coding/codecs/isac/isacfix.gypi
+++ b/third_party/webrtc/modules/audio_coding/codecs/isac/isacfix.gypi
@@ -80,7 +80,7 @@
             'WEBRTC_LINUX',
           ],
         }],
-        ['target_arch=="arm" and arm_version>=7', {
+        ['target_arch=="arm" and arm_version>=7 and arm_neon==1', {
           'sources': [
             'fix/source/lattice_armv7.S',
             'fix/source/pitch_filter_armv6.S',
@@ -124,7 +124,7 @@
     },
   ],
   'conditions': [
-    ['target_arch=="arm" and arm_version>=7 or target_arch=="arm64"', {
+    ['target_arch=="arm" and arm_version>=7 and arm_neon==1 or target_arch=="arm64"', {
       'targets': [
         {
           'target_name': 'isac_neon',
--- a/ui/base/resource/data_pack.cc
+++ b/ui/base/resource/data_pack.cc
@@ -155,9 +155,11 @@ bool DataPack::LoadImpl() {
   // 2) Verify the entries are within the appropriate bounds. There's an extra
   // entry after the last item which gives us the length of the last item.
   for (size_t i = 0; i < resource_count_ + 1; ++i) {
+    uint32 t;
     const DataPackEntry* entry = reinterpret_cast<const DataPackEntry*>(
         mmap_->data() + kHeaderLength + (i * sizeof(DataPackEntry)));
-    if (entry->file_offset > mmap_->length()) {
+    memcpy(&t, &entry->file_offset, 32/8);
+    if (t > mmap_->length()) {
       LOG(ERROR) << "Entry #" << i << " in data pack points off end of file. "
                  << "Was the file corrupted?";
       UMA_HISTOGRAM_ENUMERATION("DataPack.Load", ENTRY_NOT_FOUND,
