--- chromium-54.0.2840.59/build/toolchain/linux/BUILD.gn        2016-08-25 12:19:55.634174140 +0200
+++ chromium-54.0.2840.59/build/toolchain/linux/BUILD.gn        2016-08-25 12:19:55.634174140 +0200
@@ -6,7 +6,7 @@
 }

 clang_toolchain("clang_arm") {
-  toolprefix = "arm-linux-gnueabihf-"
+  toolprefix = ""
   toolchain_args = {
     current_cpu = "arm"
     current_os = "linux"
@@ -14,7 +14,7 @@
 }

 clang_toolchain("clang_arm64") {
-  toolprefix = "aarch64-linux-gnu-"
+  toolprefix = ""
   toolchain_args = {
     current_cpu = "arm64"
     current_os = "linux"
@@ -22,7 +22,7 @@
 }

 gcc_toolchain("arm") {
-  toolprefix = "arm-linux-gnueabihf-"
+  toolprefix = ""

   cc = "${toolprefix}gcc"
   cxx = "${toolprefix}g++"

