diff -up chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc.62~ chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc
--- chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc.62~	2024-05-06 22:56:41.000000000 +0200
+++ chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc	2024-05-09 02:09:18.318706842 +0200
@@ -359,10 +359,12 @@ StatusTray* ChromeBrowserProcessAlloy::s
   return nullptr;
 }
 
+/*
 safe_browsing::SafeBrowsingService*
 ChromeBrowserProcessAlloy::safe_browsing_service() {
   return nullptr;
 }
+*/
 
 subresource_filter::RulesetService*
 ChromeBrowserProcessAlloy::subresource_filter_ruleset_service() {
diff -up chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.h.62~ chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.h
--- chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.h.62~	2024-05-06 22:56:41.000000000 +0200
+++ chromium-124.0.6367.155/cef/libcef/browser/alloy/chrome_browser_process_alloy.h	2024-05-09 02:09:18.318706842 +0200
@@ -89,7 +89,7 @@ class ChromeBrowserProcessAlloy : public
       std::unique_ptr<BackgroundModeManager> manager) override;
 #endif
   StatusTray* status_tray() override;
-  safe_browsing::SafeBrowsingService* safe_browsing_service() override;
+  //safe_browsing::SafeBrowsingService* safe_browsing_service() override;
   subresource_filter::RulesetService* subresource_filter_ruleset_service()
       override;
   StartupData* startup_data() override;
diff -up chromium-124.0.6367.155/cef/libcef/browser/prefs/browser_prefs.cc.62~ chromium-124.0.6367.155/cef/libcef/browser/prefs/browser_prefs.cc
--- chromium-124.0.6367.155/cef/libcef/browser/prefs/browser_prefs.cc.62~	2024-05-06 22:56:41.000000000 +0200
+++ chromium-124.0.6367.155/cef/libcef/browser/prefs/browser_prefs.cc	2024-05-09 02:10:06.951197121 +0200
@@ -60,7 +60,7 @@
 #include "components/privacy_sandbox/privacy_sandbox_prefs.h"
 #include "components/proxy_config/pref_proxy_config_tracker_impl.h"
 #include "components/proxy_config/proxy_config_dictionary.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
+//#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_search_api/safe_search_util.h"
 #include "components/spellcheck/browser/pref_names.h"
 #include "components/supervised_user/core/common/buildflags.h"
@@ -275,7 +275,7 @@ std::unique_ptr<PrefService> CreatePrefS
     ProfileNetworkContextService::RegisterProfilePrefs(registry.get());
     QuietNotificationPermissionUiState::RegisterProfilePrefs(registry.get());
     RegisterBrowserViewProfilePrefs(registry.get());
-    safe_browsing::RegisterProfilePrefs(registry.get());
+    //safe_browsing::RegisterProfilePrefs(registry.get());
     unified_consent::UnifiedConsentService::RegisterPrefs(registry.get());
 
     const std::string& locale =
