diff -up chromium-120.0.6099.199/cef/patch/patch.cfg.38~ chromium-120.0.6099.199/cef/patch/patch.cfg
--- chromium-120.0.6099.199/cef/patch/patch.cfg.38~	2024-01-05 21:22:52.000000000 +0100
+++ chromium-120.0.6099.199/cef/patch/patch.cfg	2024-01-09 20:43:34.038385487 +0100
@@ -559,11 +559,12 @@ patches = [
     # https://github.com/chromiumembedded/cef/issues/2819
     'name': 'win_cpp17_msvc_sandbox_2819',
   },
-  {
+# We use system libxml, so we don't need this
+#  {
     # libxml access is now limited to targets audited by the Security Team.
     # https://chromium-review.googlesource.com/c/chromium/src/+/1884750
-    'name': 'libxml_visibility',
-  },
+#    'name': 'libxml_visibility',
+#  },
   {
     # Fix unbound AssociatedRemote error in SetBackgroundOpaque.
     # https://bugs.chromium.org/p/chromium/issues/detail?id=1070713
@@ -695,11 +696,12 @@ patches = [
     # https://github.com/chromiumembedded/cef/issues/3606
     'name': 'win_app_icon_3606'
   },
-  {
+# We use system libxml, so we don't need this
+#  {
     # linux: Disable compiler TLS in libxml2.
     # https://github.com/chromiumembedded/cef/issues/3616
-    'name': 'linux_libxml_tls_3616'
-  },
+#    'name': 'linux_libxml_tls_3616'
+#  },
   {
     # chrome: mac: Support locale configuration.
     # https://github.com/chromiumembedded/cef/issues/3623
