diff -up chromium-126.0.6478.36/cef/include/internal/cef_types.h.omv~ chromium-126.0.6478.36/cef/include/internal/cef_types.h
--- chromium-126.0.6478.36/cef/include/internal/cef_types.h.omv~	2024-06-08 15:12:54.762764476 +0200
+++ chromium-126.0.6478.36/cef/include/internal/cef_types.h	2024-06-08 15:14:24.699629431 +0200
@@ -3487,7 +3487,8 @@ typedef enum {
   CEF_CPAIT_PRICE_INSIGHTS,
   CEF_CPAIT_PRICE_READ_ANYTHING,
   CEF_CPAIT_PRODUCT_SPECIFICATIONS,
-  CEF_CPAIT_MAX_VALUE = CEF_CPAIT_PRODUCT_SPECIFICATIONS,
+  CEF_CPAIT_LENS_OVERLAY,
+  CEF_CPAIT_MAX_VALUE = CEF_CPAIT_LENS_OVERLAY,
 } cef_chrome_page_action_icon_type_t;
 
 ///
