diff -up chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc.50~ chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc
--- chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc.50~	2024-04-16 15:52:24.000000000 +0200
+++ chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.cc	2024-04-19 00:25:34.486208358 +0200
@@ -359,10 +359,12 @@ StatusTray* ChromeBrowserProcessAlloy::s
   return nullptr;
 }
 
+/*
 safe_browsing::SafeBrowsingService*
 ChromeBrowserProcessAlloy::safe_browsing_service() {
   return nullptr;
 }
+*/
 
 subresource_filter::RulesetService*
 ChromeBrowserProcessAlloy::subresource_filter_ruleset_service() {
diff -up chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.h.50~ chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.h
--- chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.h.50~	2024-04-16 15:52:24.000000000 +0200
+++ chromium-124.0.6367.60/cef/libcef/browser/alloy/chrome_browser_process_alloy.h	2024-04-19 00:25:34.486208358 +0200
@@ -89,7 +89,7 @@ class ChromeBrowserProcessAlloy : public
       std::unique_ptr<BackgroundModeManager> manager) override;
 #endif
   StatusTray* status_tray() override;
-  safe_browsing::SafeBrowsingService* safe_browsing_service() override;
+  //safe_browsing::SafeBrowsingService* safe_browsing_service() override;
   subresource_filter::RulesetService* subresource_filter_ruleset_service()
       override;
   StartupData* startup_data() override;
diff -up chromium-124.0.6367.60/cef/libcef/browser/prefs/browser_prefs.cc.50~ chromium-124.0.6367.60/cef/libcef/browser/prefs/browser_prefs.cc
--- chromium-124.0.6367.60/cef/libcef/browser/prefs/browser_prefs.cc.50~	2024-04-16 15:52:24.000000000 +0200
+++ chromium-124.0.6367.60/cef/libcef/browser/prefs/browser_prefs.cc	2024-04-19 00:38:40.802221945 +0200
@@ -59,7 +59,7 @@
 #include "components/privacy_sandbox/privacy_sandbox_prefs.h"
 #include "components/proxy_config/pref_proxy_config_tracker_impl.h"
 #include "components/proxy_config/proxy_config_dictionary.h"
-#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
+//#include "components/safe_browsing/core/common/safe_browsing_prefs.h"
 #include "components/safe_search_api/safe_search_util.h"
 #include "components/spellcheck/browser/pref_names.h"
 #include "components/supervised_user/core/common/buildflags.h"
@@ -273,7 +273,7 @@ std::unique_ptr<PrefService> CreatePrefS
     privacy_sandbox::RegisterProfilePrefs(registry.get());
     ProfileNetworkContextService::RegisterProfilePrefs(registry.get());
     RegisterBrowserViewProfilePrefs(registry.get());
-    safe_browsing::RegisterProfilePrefs(registry.get());
+    //safe_browsing::RegisterProfilePrefs(registry.get());
     unified_consent::UnifiedConsentService::RegisterPrefs(registry.get());
 
     const std::string& locale =
