--- chromium-36.0.1985.143/net/disk_cache/blockfile/entry_impl_v3.cc.omv~	2014-08-25 11:32:37.541473662 +0200
+++ chromium-36.0.1985.143/net/disk_cache/blockfile/entry_impl_v3.cc	2014-08-25 11:34:56.586735357 +0200
@@ -1391,7 +1391,7 @@ void EntryImplV3::ReportIOTime(Operation
 }
 
 void EntryImplV3::Log(const char* msg) {
-  Trace("%s 0x%p 0x%x", msg, reinterpret_cast<void*>(this), address_);
+  Trace("%s 0x%p 0x%x", msg, reinterpret_cast<void*>(this), address_.value());
   Trace("  data: 0x%x 0x%x", entry_->data_addr[0], entry_->data_addr[1]);
   Trace("  doomed: %d", doomed_);
 }
--- chromium-36.0.1985.143/third_party/WebKit/Source/core/inspector/InspectorPageAgent.cpp.omv~	2014-08-25 14:02:32.551020568 +0200
+++ chromium-36.0.1985.143/third_party/WebKit/Source/core/inspector/InspectorPageAgent.cpp	2014-08-25 14:03:14.956490427 +0200
@@ -1231,7 +1231,8 @@ bool InspectorPageAgent::applyViewportSt
     RefPtrWillBeRawPtr<StyleSheetContents> styleSheet = StyleSheetContents::create(CSSParserContext(UASheetMode, 0));
     styleSheet->parseString(String(viewportAndroidUserAgentStyleSheet, sizeof(viewportAndroidUserAgentStyleSheet)));
     OwnPtrWillBeRawPtr<RuleSet> ruleSet = RuleSet::create();
-    ruleSet->addRulesFromSheet(styleSheet.get(), MediaQueryEvaluator("screen"));
+    WebCore::MediaQueryEvaluator const screen("screen");
+    ruleSet->addRulesFromSheet(styleSheet.get(), screen);
     resolver->viewportStyleResolver()->collectViewportRules(ruleSet.get(), ViewportStyleResolver::UserAgentOrigin);
     return true;
 }
