diff -up chromium-80.0.3987.53/media/gpu/vaapi/BUILD.gn.24~ chromium-80.0.3987.53/media/gpu/vaapi/BUILD.gn
--- chromium-80.0.3987.53/media/gpu/vaapi/BUILD.gn.24~	2020-02-11 03:40:02.416009866 +0100
+++ chromium-80.0.3987.53/media/gpu/vaapi/BUILD.gn	2020-02-11 13:56:26.253588103 +0100
@@ -13,6 +13,10 @@ import("//ui/ozone/ozone.gni")
 assert(is_linux)
 assert(use_vaapi)
 
+config("vaapi_permissive") {
+  cflags = [ "-fpermissive" ]
+}
+
 generate_stubs("libva_stubs") {
   extra_header = "va_stub_header.fragment"
   sigs = [ "va.sigs" ]
@@ -101,6 +105,8 @@ source_set("vaapi") {
     "//ui/gfx/geometry",
   ]
 
+  configs += [ ":vaapi_permissive" ]
+
   public_deps = [
     "//media",
     "//skia",
