diff -up chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/input.streams/istream.rvalue chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/input.streams/istream
Only in chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/input.streams/istream: istream.assign
Only in chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/input.streams/istream: istream.cons
Only in chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/input.streams/istream: istream_sentry
diff -up chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/output.streams/ostream.rvalue chromium-55.0.2883.87/buildtools/third_party/libc++/trunk/test/std/input.output/iostream.format/output.streams/ostream
diff -up chromium-55.0.2883.87/remoting/host/fake_desktop_environment.cc.rvalue chromium-55.0.2883.87/remoting/host/fake_desktop_environment.cc
--- chromium-55.0.2883.87/remoting/host/fake_desktop_environment.cc.rvalue	2016-12-14 11:19:16.150103994 -0500
+++ chromium-55.0.2883.87/remoting/host/fake_desktop_environment.cc	2016-12-14 11:19:44.003563271 -0500
@@ -84,7 +84,7 @@ FakeDesktopEnvironment::CreateVideoCaptu
   std::unique_ptr<DesktopCapturerProxy> result(
       new DesktopCapturerProxy(capture_thread_));
   result->set_capturer(std::move(fake_capturer));
-  return result;
+  return std::move(result);
 }
 
 std::unique_ptr<webrtc::MouseCursorMonitor>
