--- chromium-62.0.3202.89/media/base/mime_util_internal.cc.0114~	2017-11-12 16:14:03.270153746 +0100
+++ chromium-62.0.3202.89/media/base/mime_util_internal.cc	2017-11-12 16:14:03.271153750 +0100
@@ -282,6 +282,7 @@ void MimeUtil::AddSupportedMediaFormats(
   matroska_audio_codecs.insert(MPEG2_AAC);
   matroska_audio_codecs.insert(MPEG4_AAC);
   matroska_audio_codecs.insert(PCM);
+  matroska_audio_codecs.insert(FLAC);
   
   CodecSet matroska_video_codecs;
   matroska_video_codecs.insert(THEORA);
