--- chromium-61.0.3124.4/tools/gn/BUILD.gn.0021~	2017-06-13 18:06:33.286161143 +0200
+++ chromium-61.0.3124.4/tools/gn/BUILD.gn	2017-06-13 18:07:37.969672191 +0200
@@ -268,7 +268,6 @@ executable("gn") {
 
   deps = [
     ":gn_lib",
-    ":last_commit_position",
     "//base",
     "//build/config:exe_and_shlib_deps",
     "//build/win:default_exe_manifest",
--- chromium-61.0.3124.4/tools/gn/gn_main.cc.0021~	2017-06-10 03:03:13.000000000 +0200
+++ chromium-61.0.3124.4/tools/gn/gn_main.cc	2017-06-13 18:06:33.287161120 +0200
@@ -12,13 +12,7 @@
 #include "tools/gn/standard_out.h"
 #include "tools/gn/switches.h"
 
-// Only the GN-generated build makes this header for now.
-// TODO(brettw) consider adding this if we need it in GYP.
-#if defined(GN_BUILD)
-#include "tools/gn/last_commit_position.h"
-#else
 #define LAST_COMMIT_POSITION "UNKNOWN"
-#endif
 
 namespace {
 
