diff --git a/v8/tools/gyp/v8.gyp b/v8/tools/gyp/v8.gyp
index c57c52b..1c8ba82 100644
--- a/v8/tools/gyp/v8.gyp
+++ b/v8/tools/gyp/v8.gyp
@@ -1004,7 +1004,7 @@
           'include_dirs++': [
             '<(shim_headers_path)',
           ],
-          'direct_dependent_settings': {
+          'all_dependent_settings': {
             'include_dirs+++': [
               '<(shim_headers_path)',
             ],
