diff -up chromium-66.0.3359.117/build/config/clang/clang.gni.system-clang chromium-66.0.3359.117/build/config/clang/clang.gni
--- chromium-66.0.3359.117/build/config/clang/clang.gni.system-clang	2018-04-18 12:08:45.658190297 -0400
+++ chromium-66.0.3359.117/build/config/clang/clang.gni	2018-04-18 12:09:37.612032982 -0400
@@ -4,7 +4,7 @@
 
 import("//build/toolchain/toolchain.gni")
 
-default_clang_base_path = "//third_party/llvm-build/Release+Asserts"
+default_clang_base_path = "/usr"
 
 declare_args() {
   # Indicates if the build should use the Chrome-specific plugins for enforcing
