diff -up chromium-56.0.2924.87/remoting/host/security_key/fake_security_key_ipc_server.cc.unique-ptr-fix chromium-56.0.2924.87/remoting/host/security_key/fake_security_key_ipc_server.cc
--- chromium-56.0.2924.87/remoting/host/security_key/fake_security_key_ipc_server.cc.unique-ptr-fix	2017-02-24 07:50:09.183551244 -0500
+++ chromium-56.0.2924.87/remoting/host/security_key/fake_security_key_ipc_server.cc	2017-02-24 07:50:31.619111443 -0500
@@ -130,7 +130,7 @@ std::unique_ptr<SecurityKeyIpcServer> Fa
 
   ipc_server_map_[connection_id] = fake_ipc_server->AsWeakPtr();
 
-  return fake_ipc_server;
+  return std::move(fake_ipc_server);
 }
 
 base::WeakPtr<FakeSecurityKeyIpcServer>
