--- chromium-65.0.3315.3/base/synchronization/lock_impl_posix.cc.0008~	2018-01-09 21:06:20.000000000 +0100
+++ chromium-65.0.3315.3/base/synchronization/lock_impl_posix.cc	2018-01-10 22:57:34.601438098 +0100
@@ -8,7 +8,7 @@
 
 #include "base/debug/activity_tracker.h"
 #include "base/synchronization/lock.h"
-#include "base/synchronization/synchronization_buildflags.h"
+//#include "base/synchronization/synchronization_buildflags.h"
 #include "build/build_config.h"
 
 namespace base {
@@ -78,7 +78,7 @@ void LockImpl::Lock() {
 
 // static
 bool LockImpl::PriorityInheritanceAvailable() {
-#if BUILDFLAG(ENABLE_MUTEX_PRIORITY_INHERITANCE)
+#if 1
   return true;
 #elif PRIORITY_INHERITANCE_LOCKS_POSSIBLE() && defined(OS_MACOSX)
   return true;
