diff -up chromium-89.0.4389.47/net/third_party/quiche/src/quic/core/quic_path_validator.cc.omv~ chromium-89.0.4389.47/net/third_party/quiche/src/quic/core/quic_path_validator.cc
--- chromium-89.0.4389.47/net/third_party/quiche/src/quic/core/quic_path_validator.cc.omv~	2021-02-12 14:30:35.072615942 +0100
+++ chromium-89.0.4389.47/net/third_party/quiche/src/quic/core/quic_path_validator.cc	2021-02-12 14:30:38.863587716 +0100
@@ -68,7 +68,7 @@ void QuicPathValidator::OnPathResponse(c
 void QuicPathValidator::StartPathValidation(
     std::unique_ptr<QuicPathValidationContext> context,
     std::unique_ptr<ResultDelegate> result_delegate) {
-  DCHECK_NE(nullptr, context);
+  //DCHECK_NE(nullptr, context);
   QUIC_DLOG(INFO) << "Start validating path " << *context
                   << " via writer: " << context->WriterToUse();
   if (path_context_ != nullptr) {
