diff -up chromium-124.0.6367.60/build/config/compiler/BUILD.gn.40~ chromium-124.0.6367.60/build/config/compiler/BUILD.gn
--- chromium-124.0.6367.60/build/config/compiler/BUILD.gn.40~	2024-04-15 22:33:43.985039200 +0200
+++ chromium-124.0.6367.60/build/config/compiler/BUILD.gn	2024-04-19 00:11:11.880507555 +0200
@@ -734,26 +734,26 @@ config("compiler") {
       "-fsplit-lto-unit",
     ]
 
-    if (thin_lto_enable_cache) {
+    #if (thin_lto_enable_cache) {
       # Limit the size of the ThinLTO cache to the lesser of 10% of
       # available disk space, 40GB and 100000 files.
-      cache_policy =
-          "cache_size=10%:cache_size_bytes=40g:cache_size_files=100000"
-      cache_dir = rebase_path("$root_out_dir/thinlto-cache", root_build_dir)
-      if (is_win) {
-        ldflags += [
-          "/lldltocache:$cache_dir",
-          "/lldltocachepolicy:$cache_policy",
-        ]
-      } else {
-        if (is_apple) {
-          ldflags += [ "-Wl,-cache_path_lto,$cache_dir" ]
-        } else {
-          ldflags += [ "-Wl,--thinlto-cache-dir=$cache_dir" ]
-        }
-        ldflags += [ "-Wl,--thinlto-cache-policy=$cache_policy" ]
-      }
-    }
+      #cache_policy =
+      #    "cache_size=10%:cache_size_bytes=40g:cache_size_files=100000"
+      #cache_dir = rebase_path("$root_out_dir/thinlto-cache", root_build_dir)
+      #if (is_win) {
+      #  ldflags += [
+      #    "/lldltocache:$cache_dir",
+      #    "/lldltocachepolicy:$cache_policy",
+      #  ]
+      #} else {
+      #  if (is_apple) {
+      #    ldflags += [ "-Wl,-cache_path_lto,$cache_dir" ]
+      #  } else {
+      #    ldflags += [ "-Wl,--thinlto-cache-dir=$cache_dir" ]
+      #  }
+      #  ldflags += [ "-Wl,--thinlto-cache-policy=$cache_policy" ]
+      #}
+    #}
 
     # An import limit of 30 has better performance (per speedometer) and lower
     # binary size than the default setting of 100.
@@ -2182,8 +2182,6 @@ config("thin_archive") {
   # archives.
   if ((is_posix && !is_nacl && (!is_apple || use_lld)) || is_fuchsia) {
     arflags = [ "-T" ]
-  } else if (is_win && use_lld) {
-    arflags = [ "/llvmlibthin" ]
   }
 }
 
