diff -up chromium-85.0.4164.2/third_party/webrtc/rtc_base/physical_socket_server.h.21~ chromium-85.0.4164.2/third_party/webrtc/rtc_base/physical_socket_server.h
--- chromium-85.0.4164.2/third_party/webrtc/rtc_base/physical_socket_server.h.21~	2020-06-06 21:46:54.580693608 +0200
+++ chromium-85.0.4164.2/third_party/webrtc/rtc_base/physical_socket_server.h	2020-06-06 22:45:32.047822449 +0200
@@ -16,6 +16,11 @@
 #define WEBRTC_USE_EPOLL 1
 #endif
 
+// for SIOCGSTAMP in Linux 5.2
+#if defined(WEBRTC_LINUX)
+#include <linux/sockios.h>
+#endif
+
 #include <array>
 #include <memory>
 #include <set>
