diff -up chromium-90.0.4430.61/components/translate/core/language_detection/ngram_hash_ops_utils.cc.omv~ chromium-90.0.4430.61/components/translate/core/language_detection/ngram_hash_ops_utils.cc
--- chromium-90.0.4430.61/components/translate/core/language_detection/ngram_hash_ops_utils.cc.omv~	2021-04-12 22:16:14.272000386 +0200
+++ chromium-90.0.4430.61/components/translate/core/language_detection/ngram_hash_ops_utils.cc	2021-04-12 22:16:21.558660685 +0200
@@ -5,6 +5,7 @@
 #include "components/translate/core/language_detection/ngram_hash_ops_utils.h"
 
 #include <vector>
+#include <cstring>
 
 #include "third_party/utf/src/include/utf.h"
 
diff -up chromium-90.0.4430.61/components/translate/core/language_detection/quantization_utils.h.omv~ chromium-90.0.4430.61/components/translate/core/language_detection/quantization_utils.h
--- chromium-90.0.4430.61/components/translate/core/language_detection/quantization_utils.h.omv~	2021-04-12 22:07:14.952301100 +0200
+++ chromium-90.0.4430.61/components/translate/core/language_detection/quantization_utils.h	2021-04-12 22:07:24.205633289 +0200
@@ -7,6 +7,9 @@
 
 #include <algorithm>
 #include <cmath>
+extern "C" {
+#include <stdint.h>
+}
 
 namespace translate {
 
