diff -up chromium-93.0.4577.42/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc.omv~ chromium-93.0.4577.42/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc
--- chromium-93.0.4577.42/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc.omv~	2021-08-15 06:12:02.482288779 +0200
+++ chromium-93.0.4577.42/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc	2021-08-15 06:12:12.753222019 +0200
@@ -138,7 +138,7 @@ void InstallAlternateStackLocked() {
   // SIGSTKSZ may be too small to prevent the signal handlers from overrunning
   // the alternative stack. Ensure that the size of the alternative stack is
   // large enough.
-  static const unsigned kSigStackSize = std::max(16384, SIGSTKSZ);
+  static const unsigned kSigStackSize = std::max<unsigned>(16384, SIGSTKSZ);
 
   // Only set an alternative stack if there isn't already one, or if the current
   // one is too small.
