diff -up chromium-124.0.6367.60/third_party/blink/renderer/modules/webgpu/gpu_adapter_info.h.omv~ chromium-124.0.6367.60/third_party/blink/renderer/modules/webgpu/gpu_adapter_info.h
--- chromium-124.0.6367.60/third_party/blink/renderer/modules/webgpu/gpu_adapter_info.h.omv~	2024-04-19 15:56:41.152152693 +0200
+++ chromium-124.0.6367.60/third_party/blink/renderer/modules/webgpu/gpu_adapter_info.h	2024-04-19 15:56:48.064217772 +0200
@@ -8,6 +8,7 @@
 #include "third_party/blink/renderer/platform/bindings/script_wrappable.h"
 #include "third_party/blink/renderer/platform/heap/collection_support/heap_vector.h"
 #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
+#include <optional>
 
 namespace blink {
 
diff -up chromium-124.0.6367.60/base/containers/span_reader.h.12~ chromium-124.0.6367.60/base/containers/span_reader.h
--- chromium-124.0.6367.60/base/containers/span_reader.h.12~	2024-04-15 22:33:42.645014300 +0200
+++ chromium-124.0.6367.60/base/containers/span_reader.h	2024-04-19 12:49:28.098411326 +0200
@@ -7,6 +7,7 @@
 
 #include "base/containers/span.h"
 #include "base/numerics/safe_conversions.h"
+#include <optional>
 
 namespace base {
 
diff -up chromium-124.0.6367.60/content/browser/first_party_sets/first_party_sets_handler_database_helper.h.12~ chromium-124.0.6367.60/content/browser/first_party_sets/first_party_sets_handler_database_helper.h
--- chromium-124.0.6367.60/content/browser/first_party_sets/first_party_sets_handler_database_helper.h.12~	2024-04-15 22:33:57.374096000 +0200
+++ chromium-124.0.6367.60/content/browser/first_party_sets/first_party_sets_handler_database_helper.h	2024-04-19 12:49:28.102411368 +0200
@@ -9,6 +9,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <optional>
 
 #include "base/containers/flat_set.h"
 #include "base/files/file_path.h"
diff -up chromium-124.0.6367.60/media/filters/temporal_scalability_id_extractor.cc.12~ chromium-124.0.6367.60/media/filters/temporal_scalability_id_extractor.cc
--- chromium-124.0.6367.60/media/filters/temporal_scalability_id_extractor.cc.12~	2024-04-15 22:34:01.102369800 +0200
+++ chromium-124.0.6367.60/media/filters/temporal_scalability_id_extractor.cc	2024-04-19 12:49:28.102411368 +0200
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #include "media/filters/temporal_scalability_id_extractor.h"
+#include <bitset>
 
 namespace media {
 
diff -up chromium-124.0.6367.60/third_party/angle/src/libANGLE/renderer/vulkan/FramebufferVk.cpp.12~ chromium-124.0.6367.60/third_party/angle/src/libANGLE/renderer/vulkan/FramebufferVk.cpp
--- chromium-124.0.6367.60/third_party/angle/src/libANGLE/renderer/vulkan/FramebufferVk.cpp.12~	2024-04-15 22:35:21.680214000 +0200
+++ chromium-124.0.6367.60/third_party/angle/src/libANGLE/renderer/vulkan/FramebufferVk.cpp	2024-04-19 12:49:28.102411368 +0200
@@ -1726,10 +1726,10 @@ angle::Result FramebufferVk::generateFra
                 for (uint32_t point = 0; point < activeFocalPoints.size(); point++)
                 {
                     float density =
-                        1.0f / std::max(std::powf(activeFocalPoints[point].focalX - px, 2) *
-                                                std::powf(activeFocalPoints[point].gainX, 2) +
-                                            std::powf(activeFocalPoints[point].focalY - py, 2) *
-                                                std::powf(activeFocalPoints[point].gainY, 2) -
+                        1.0f / std::max(powf(activeFocalPoints[point].focalX - px, 2) *
+                                                powf(activeFocalPoints[point].gainX, 2) +
+                                            powf(activeFocalPoints[point].focalY - py, 2) *
+                                                powf(activeFocalPoints[point].gainY, 2) -
                                             activeFocalPoints[point].foveaArea,
                                         1.0f);
 
