diff -up chromium-135.0.7049.84/build/config/clang/BUILD.gn.58~ chromium-135.0.7049.84/build/config/clang/BUILD.gn
--- chromium-135.0.7049.84/build/config/clang/BUILD.gn.58~	2025-04-07 23:40:33.000000000 +0200
+++ chromium-135.0.7049.84/build/config/clang/BUILD.gn	2025-04-09 21:25:20.344525887 +0200
@@ -166,8 +166,8 @@ template("clang_lib") {
       _prefix = "lib"
       _suffix = ""
       _ext = "a"
+      _libprefix = ""
 
-      _clang_lib_dir = "$clang_base_path/lib/clang/$clang_version/lib"
       if (is_win) {
         _dir = "windows"
         _prefix = ""
@@ -197,7 +197,24 @@ template("clang_lib") {
         } else {
           assert(false)  # Unhandled cpu type
         }
-      } else if (is_linux || is_chromeos) {
+      } else if (is_linux) {
+        if (current_cpu == "x64") {
+          _dir = "x86_64-pc-linux-gnu"
+          _libprefix = "64"
+        } else if (current_cpu == "x86") {
+          _dir = "i386-pc-linux-gnu"
+        } else if (current_cpu == "arm") {
+          _dir = "armv7-unknown-linux-gnueabihf"
+        } else if (current_cpu == "arm64") {
+          _dir = "aarch64-unknown-linux-gnu"
+          _libprefix = "64"
+        } else if (current_cpu == "ppc64") {
+          _dir = "powerpc64le-pc-linux-gnu"
+          _libprefix = "64"
+        } else {
+          assert(false)  # Unhandled cpu type
+        }
+      } else if (is_chromeos) {
         if (current_cpu == "x64") {
           _dir = "x86_64-unknown-linux-gnu"
         } else if (current_cpu == "x86") {
@@ -235,6 +252,10 @@ template("clang_lib") {
       } else {
         assert(false)  # Unhandled target platform
       }
+      _clang_lib_dir = "$clang_base_path/lib${_libprefix}/clang/$clang_version/lib"
+      if (_use_cros_sysroot_libs) {
+        _clang_lib_dir = _cros_resource_dir
+      }
 
       _lib_file = "${_prefix}clang_rt.${_libname}${_suffix}.${_ext}"
       libs = [ "$_clang_lib_dir/$_dir/$_lib_file" ]
