diff -up chromium-117.0.5938.62/chrome/common/chrome_paths.cc.4~ chromium-117.0.5938.62/chrome/common/chrome_paths.cc
--- chromium-117.0.5938.62/chrome/common/chrome_paths.cc.4~	2023-09-14 01:03:30.714378971 +0200
+++ chromium-117.0.5938.62/chrome/common/chrome_paths.cc	2023-09-14 01:06:28.324920469 +0200
@@ -376,6 +376,16 @@ bool PathProvider(int key, base::FilePat
 
 #if BUILDFLAG(ENABLE_WIDEVINE)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      base::PathService::Get(base::DIR_HOME, &cur);
+      cur = cur.Append(FILE_PATH_LITERAL(".local/lib/libwidevinecdm.so"));
+      if (base::PathExists(cur)) {
+        break;
+      }
+      // Yes, this has an arch hardcoded in the path, but at this time, it is the only place to find libwidevinecdm.so
+      if (base::PathExists(base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so")))) {
+        cur = base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so"));
+        break;
+      }
       if (!GetComponentDirectory(&cur)) {
         return false;
       }
