diff -up chromium-124.0.6367.155/chrome/browser/devtools/BUILD.gn.17~ chromium-124.0.6367.155/chrome/browser/devtools/BUILD.gn
--- chromium-124.0.6367.155/chrome/browser/devtools/BUILD.gn.17~	2024-05-09 22:16:35.199838703 +0200
+++ chromium-124.0.6367.155/chrome/browser/devtools/BUILD.gn	2024-05-09 22:17:27.820476430 +0200
@@ -113,6 +113,7 @@ static_library("devtools") {
     "//chrome/browser/autofill:autofill",
     "//components/autofill/content/browser:browser",
     "//components/autofill/core/browser:browser",
+    "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags:buildflags",
     "//content/public/browser",
     "//net",
