diff -up chromium-124.0.6367.60/components/search_engines/search_engine_countries-inc.cc.omv~ chromium-124.0.6367.60/components/search_engines/search_engine_countries-inc.cc
--- chromium-124.0.6367.60/components/search_engines/search_engine_countries-inc.cc.omv~	2024-04-18 23:01:17.827854902 +0200
+++ chromium-124.0.6367.60/components/search_engines/search_engine_countries-inc.cc	2024-04-18 23:29:03.932026904 +0200
@@ -33,6 +33,7 @@ struct EngineAndTier {
 
 // Default (for countries with no better engine set)
 constexpr EngineAndTier engines_default[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -44,6 +45,7 @@ constexpr EngineAndTier engines_default[
 // clang-format off
 // United Arab Emirates
 constexpr EngineAndTier engines_AE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -53,24 +55,25 @@ constexpr EngineAndTier engines_AE[] = {
 
 // Albania
 constexpr EngineAndTier engines_AL[] = {
+    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Argentina
 constexpr EngineAndTier engines_AR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_ar},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Austria
 constexpr EngineAndTier engines_AT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &brave},
@@ -83,6 +86,7 @@ constexpr EngineAndTier engines_AT[] = {
 
 // Australia
 constexpr EngineAndTier engines_AU[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_au},
@@ -92,6 +96,7 @@ constexpr EngineAndTier engines_AU[] = {
 
 // Bosnia and Herzegovina
 constexpr EngineAndTier engines_BA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -101,6 +106,7 @@ constexpr EngineAndTier engines_BA[] = {
 
 // Belgium
 constexpr EngineAndTier engines_BE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &brave},
     {SearchEngineTier::kTopEngines, &bing},
@@ -113,6 +119,7 @@ constexpr EngineAndTier engines_BE[] = {
 
 // Bulgaria
 constexpr EngineAndTier engines_BG[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -125,53 +132,53 @@ constexpr EngineAndTier engines_BG[] = {
 
 // Bahrain
 constexpr EngineAndTier engines_BH[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Burundi
 constexpr EngineAndTier engines_BI[] = {
+    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Brunei
 constexpr EngineAndTier engines_BN[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Bolivia
 constexpr EngineAndTier engines_BO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Brazil
 constexpr EngineAndTier engines_BR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_br},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Belarus
 constexpr EngineAndTier engines_BY[] = {
-    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yandex_by},
+    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &mail_ru},
@@ -179,43 +186,53 @@ constexpr EngineAndTier engines_BY[] = {
 
 // Belize
 constexpr EngineAndTier engines_BZ[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Canada
 constexpr EngineAndTier engines_CA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_ca},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Switzerland
 constexpr EngineAndTier engines_CH[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &yahoo_ch},
     {SearchEngineTier::kTopEngines, &ecosia},
+    {SearchEngineTier::kRemainingEngines, &qwant},
+    {SearchEngineTier::kRemainingEngines, &brave},
+    {SearchEngineTier::kRemainingEngines, &panda},
+    {SearchEngineTier::kRemainingEngines, &metager_de},
+    {SearchEngineTier::kRemainingEngines, &oceanhero},
+    {SearchEngineTier::kRemainingEngines, &karma},
+    {SearchEngineTier::kRemainingEngines, &privacywall},
+    {SearchEngineTier::kRemainingEngines, &lilo},
 };
 
 // Chile
 constexpr EngineAndTier engines_CL[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_cl},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // China
 constexpr EngineAndTier engines_CN[] = {
     {SearchEngineTier::kTopEngines, &baidu},
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &sogou},
     {SearchEngineTier::kTopEngines, &so_360},
@@ -224,6 +246,7 @@ constexpr EngineAndTier engines_CN[] = {
 
 // Colombia
 constexpr EngineAndTier engines_CO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_co},
@@ -233,15 +256,16 @@ constexpr EngineAndTier engines_CO[] = {
 
 // Costa Rica
 constexpr EngineAndTier engines_CR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Republic of Cyprus
 constexpr EngineAndTier engines_CY[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -254,6 +278,7 @@ constexpr EngineAndTier engines_CY[] = {
 
 // Czech Republic
 constexpr EngineAndTier engines_CZ[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &seznam_cz},
     {SearchEngineTier::kTopEngines, &bing},
@@ -266,6 +291,7 @@ constexpr EngineAndTier engines_CZ[] = {
 
 // Germany
 constexpr EngineAndTier engines_DE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &ecosia},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -278,6 +304,7 @@ constexpr EngineAndTier engines_DE[] = {
 
 // Denmark
 constexpr EngineAndTier engines_DK[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -291,24 +318,25 @@ constexpr EngineAndTier engines_DK[] = {
 
 // Dominican Republic
 constexpr EngineAndTier engines_DO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Algeria
 constexpr EngineAndTier engines_DZ[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_fr},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Ecuador
 constexpr EngineAndTier engines_EC[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_es},
@@ -318,6 +346,7 @@ constexpr EngineAndTier engines_EC[] = {
 
 // Estonia
 constexpr EngineAndTier engines_EE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -330,15 +359,16 @@ constexpr EngineAndTier engines_EE[] = {
 
 // Egypt
 constexpr EngineAndTier engines_EG[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Spain
 constexpr EngineAndTier engines_ES[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &brave},
     {SearchEngineTier::kTopEngines, &bing},
@@ -351,6 +381,7 @@ constexpr EngineAndTier engines_ES[] = {
 
 // Finland
 constexpr EngineAndTier engines_FI[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &brave},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -363,6 +394,7 @@ constexpr EngineAndTier engines_FI[] = {
 
 // Faroe Islands
 constexpr EngineAndTier engines_FO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_uk},
@@ -372,6 +404,7 @@ constexpr EngineAndTier engines_FO[] = {
 
 // France
 constexpr EngineAndTier engines_FR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &brave},
     {SearchEngineTier::kTopEngines, &ecosia},
@@ -384,6 +417,7 @@ constexpr EngineAndTier engines_FR[] = {
 
 // United Kingdom
 constexpr EngineAndTier engines_GB[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_uk},
@@ -393,6 +427,7 @@ constexpr EngineAndTier engines_GB[] = {
 
 // Greece
 constexpr EngineAndTier engines_GR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -405,15 +440,16 @@ constexpr EngineAndTier engines_GR[] = {
 
 // Guatemala
 constexpr EngineAndTier engines_GT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Hong Kong
 constexpr EngineAndTier engines_HK[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_hk},
@@ -423,15 +459,16 @@ constexpr EngineAndTier engines_HK[] = {
 
 // Honduras
 constexpr EngineAndTier engines_HN[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Croatia
 constexpr EngineAndTier engines_HR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -444,6 +481,7 @@ constexpr EngineAndTier engines_HR[] = {
 
 // Hungary
 constexpr EngineAndTier engines_HU[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -456,8 +494,8 @@ constexpr EngineAndTier engines_HU[] = {
 
 // Indonesia
 constexpr EngineAndTier engines_ID[] = {
-    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yandex_com},
+    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_id},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -465,6 +503,7 @@ constexpr EngineAndTier engines_ID[] = {
 
 // Ireland
 constexpr EngineAndTier engines_IE[] = {
+  {SearchEngineTier::kTopEngines, &yandex_com},
   {SearchEngineTier::kTopEngines, &google},
   {SearchEngineTier::kTopEngines, &duckduckgo},
   {SearchEngineTier::kTopEngines, &brave},
@@ -477,6 +516,7 @@ constexpr EngineAndTier engines_IE[] = {
 
 // Israel
 constexpr EngineAndTier engines_IL[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yandex_ru},
@@ -486,15 +526,16 @@ constexpr EngineAndTier engines_IL[] = {
 
 // India
 constexpr EngineAndTier engines_IN[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_in},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Iraq
 constexpr EngineAndTier engines_IQ[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yandex_tr},
@@ -504,15 +545,16 @@ constexpr EngineAndTier engines_IQ[] = {
 
 // Iran
 constexpr EngineAndTier engines_IR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Iceland
 constexpr EngineAndTier engines_IS[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -525,6 +567,7 @@ constexpr EngineAndTier engines_IS[] = {
 
 // Italy
 constexpr EngineAndTier engines_IT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &brave},
@@ -537,6 +580,7 @@ constexpr EngineAndTier engines_IT[] = {
 
 // Jamaica
 constexpr EngineAndTier engines_JM[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -546,6 +590,7 @@ constexpr EngineAndTier engines_JM[] = {
 
 // Jordan
 constexpr EngineAndTier engines_JO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -555,6 +600,7 @@ constexpr EngineAndTier engines_JO[] = {
 
 // Japan
 constexpr EngineAndTier engines_JP[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yahoo_jp},
     {SearchEngineTier::kTopEngines, &bing},
@@ -564,15 +610,16 @@ constexpr EngineAndTier engines_JP[] = {
 
 // Kenya
 constexpr EngineAndTier engines_KE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // South Korea
 constexpr EngineAndTier engines_KR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &naver},
     {SearchEngineTier::kTopEngines, &bing},
@@ -582,17 +629,17 @@ constexpr EngineAndTier engines_KR[] = {
 
 // Kuwait
 constexpr EngineAndTier engines_KW[] = {
+    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
-    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Kazakhstan
 constexpr EngineAndTier engines_KZ[] = {
-    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yandex_kz},
+    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &mail_ru},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -600,15 +647,16 @@ constexpr EngineAndTier engines_KZ[] = {
 
 // Lebanon
 constexpr EngineAndTier engines_LB[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Liechtenstein
 constexpr EngineAndTier engines_LI[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -621,6 +669,7 @@ constexpr EngineAndTier engines_LI[] = {
 
 // Lithuania
 constexpr EngineAndTier engines_LT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -633,6 +682,7 @@ constexpr EngineAndTier engines_LT[] = {
 
 // Luxembourg
 constexpr EngineAndTier engines_LU[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -645,6 +695,7 @@ constexpr EngineAndTier engines_LU[] = {
 
 // Latvia
 constexpr EngineAndTier engines_LV[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -657,35 +708,35 @@ constexpr EngineAndTier engines_LV[] = {
 
 // Libya
 constexpr EngineAndTier engines_LY[] = {
+    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
-    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Morocco
 constexpr EngineAndTier engines_MA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_fr},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Monaco
 constexpr EngineAndTier engines_MC[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_fr},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Moldova
 constexpr EngineAndTier engines_MD[] = {
-    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yandex_ru},
+    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &mail_ru},
@@ -693,24 +744,25 @@ constexpr EngineAndTier engines_MD[] = {
 
 // Montenegro
 constexpr EngineAndTier engines_ME[] = {
+    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
-    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Macedonia
 constexpr EngineAndTier engines_MK[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Malta
 constexpr EngineAndTier engines_MT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &brave},
@@ -723,6 +775,7 @@ constexpr EngineAndTier engines_MT[] = {
 
 // Mexico
 constexpr EngineAndTier engines_MX[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_mx},
@@ -732,24 +785,25 @@ constexpr EngineAndTier engines_MX[] = {
 
 // Malaysia
 constexpr EngineAndTier engines_MY[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_my},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Nicaragua
 constexpr EngineAndTier engines_NI[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Netherlands
 constexpr EngineAndTier engines_NL[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -762,6 +816,7 @@ constexpr EngineAndTier engines_NL[] = {
 
 // Norway
 constexpr EngineAndTier engines_NO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -774,6 +829,7 @@ constexpr EngineAndTier engines_NO[] = {
 
 // New Zealand
 constexpr EngineAndTier engines_NZ[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_nz},
@@ -783,28 +839,28 @@ constexpr EngineAndTier engines_NZ[] = {
 
 // Oman
 constexpr EngineAndTier engines_OM[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Panama
 constexpr EngineAndTier engines_PA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_es},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Peru
 constexpr EngineAndTier engines_PE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_pe},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
@@ -828,6 +884,7 @@ constexpr EngineAndTier engines_PK[] = {
 
 // Poland
 constexpr EngineAndTier engines_PL[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &brave},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -840,15 +897,16 @@ constexpr EngineAndTier engines_PL[] = {
 
 // Puerto Rico
 constexpr EngineAndTier engines_PR[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Portugal
 constexpr EngineAndTier engines_PT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &brave},
     {SearchEngineTier::kTopEngines, &bing},
@@ -861,24 +919,25 @@ constexpr EngineAndTier engines_PT[] = {
 
 // Paraguay
 constexpr EngineAndTier engines_PY[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_es},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Qatar
 constexpr EngineAndTier engines_QA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_tr},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_tr},
 };
 
 // Romania
 constexpr EngineAndTier engines_RO[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yahoo_emea},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -891,10 +950,10 @@ constexpr EngineAndTier engines_RO[] = {
 
 // Serbia
 constexpr EngineAndTier engines_RS[] = {
+    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
@@ -909,24 +968,25 @@ constexpr EngineAndTier engines_RU[] = {
 
 // Rwanda
 constexpr EngineAndTier engines_RW[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Saudi Arabia
 constexpr EngineAndTier engines_SA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Sweden
 constexpr EngineAndTier engines_SE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -939,15 +999,16 @@ constexpr EngineAndTier engines_SE[] = {
 
 // Singapore
 constexpr EngineAndTier engines_SG[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_sg},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Slovenia
 constexpr EngineAndTier engines_SI[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -960,6 +1021,7 @@ constexpr EngineAndTier engines_SI[] = {
 
 // Slovakia
 constexpr EngineAndTier engines_SK[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
@@ -972,44 +1034,44 @@ constexpr EngineAndTier engines_SK[] = {
 
 // El Salvador
 constexpr EngineAndTier engines_SV[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_es},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Syria
 constexpr EngineAndTier engines_SY[] = {
+    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
-    {SearchEngineTier::kTopEngines, &yandex_ru},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Thailand
 constexpr EngineAndTier engines_TH[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_th},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Tunisia
 constexpr EngineAndTier engines_TN[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_fr},
-    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Turkey
 constexpr EngineAndTier engines_TR[] = {
-    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yandex_tr},
+    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yahoo_tr},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
@@ -1017,15 +1079,16 @@ constexpr EngineAndTier engines_TR[] = {
 
 // Trinidad and Tobago
 constexpr EngineAndTier engines_TT[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Taiwan
 constexpr EngineAndTier engines_TW[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yahoo_tw},
     {SearchEngineTier::kTopEngines, &bing},
@@ -1035,17 +1098,17 @@ constexpr EngineAndTier engines_TW[] = {
 
 // Tanzania
 constexpr EngineAndTier engines_TZ[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Ukraine
 constexpr EngineAndTier engines_UA[] = {
-    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &yandex_ru},
+    {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &duckduckgo},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -1053,33 +1116,34 @@ constexpr EngineAndTier engines_UA[] = {
 
 // United States
 constexpr EngineAndTier engines_US[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Uruguay
 constexpr EngineAndTier engines_UY[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_es},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Venezuela
 constexpr EngineAndTier engines_VE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo_es},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Vietnam
 constexpr EngineAndTier engines_VN[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &coccoc},
     {SearchEngineTier::kTopEngines, &yahoo},
@@ -1089,24 +1153,25 @@ constexpr EngineAndTier engines_VN[] = {
 
 // Yemen
 constexpr EngineAndTier engines_YE[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // South Africa
 constexpr EngineAndTier engines_ZA[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
     {SearchEngineTier::kTopEngines, &duckduckgo},
-    {SearchEngineTier::kTopEngines, &yandex_com},
 };
 
 // Zimbabwe
 constexpr EngineAndTier engines_ZW[] = {
+    {SearchEngineTier::kTopEngines, &yandex_com},
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
