--- chromium-64.0.3253.3/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h.omv~	2017-11-06 03:12:12.165490461 +0100
+++ chromium-64.0.3253.3/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h	2017-11-06 03:12:20.899515168 +0100
@@ -37,7 +37,7 @@
 #include <map>
 
 #include "common/using_std_string.h"
-#include "third_party/curl/curl.h"
+#include <curl/curl.h>
 
 namespace google_breakpad {
 class LibcurlWrapper {
