diff -up chromium-75.0.3770.80/components/omnibox/browser/autocomplete_match.cc.revert-daff6b chromium-75.0.3770.80/components/omnibox/browser/autocomplete_match.cc
--- chromium-75.0.3770.80/components/omnibox/browser/autocomplete_match.cc.revert-daff6b	2019-06-06 14:54:13.819190586 -0400
+++ chromium-75.0.3770.80/components/omnibox/browser/autocomplete_match.cc	2019-06-06 14:54:31.892839991 -0400
@@ -154,8 +154,7 @@ AutocompleteMatch::AutocompleteMatch(con
       additional_info(match.additional_info),
       duplicate_matches(match.duplicate_matches) {}
 
-AutocompleteMatch::AutocompleteMatch(AutocompleteMatch&& match) noexcept =
-    default;
+AutocompleteMatch::AutocompleteMatch(AutocompleteMatch&& match) = default;
 
 AutocompleteMatch::~AutocompleteMatch() {
 }
