diff -up chromium-92.0.4515.131/chrome/common/chrome_paths.cc.9~ chromium-92.0.4515.131/chrome/common/chrome_paths.cc
--- chromium-92.0.4515.131/chrome/common/chrome_paths.cc.9~	2021-08-14 04:38:30.267041808 +0200
+++ chromium-92.0.4515.131/chrome/common/chrome_paths.cc	2021-08-14 04:42:16.501598294 +0200
@@ -313,6 +313,15 @@ bool PathProvider(int key, base::FilePat
 
 #if BUILDFLAG(ENABLE_WIDEVINE)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      base::PathService::Get(base::DIR_HOME, &cur);
+      cur = cur.Append(FILE_PATH_LITERAL(".local/lib/libwidevinecdm.so"));
+      if (base::PathExists(cur)) {
+        break;
+      }
+      if (base::PathExists(base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/libwidevinecdm.so")))) {
+        cur = base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/libwidevinecdm.so"));
+        break;
+      }
       if (!GetComponentDirectory(&cur))
         return false;
 #if !BUILDFLAG(IS_CHROMEOS_ASH)
