diff -up chromium-69.0.3497.100/media/base/mime_util_internal.cc.38~ chromium-69.0.3497.100/media/base/mime_util_internal.cc
--- chromium-69.0.3497.100/media/base/mime_util_internal.cc.38~	2018-10-02 22:33:23.202267666 +0200
+++ chromium-69.0.3497.100/media/base/mime_util_internal.cc	2018-10-02 22:35:18.081936900 +0200
@@ -76,6 +76,11 @@ const base::flat_map<std::string, MimeUt
                 {"vp8.0", MimeUtil::VP8}, {"theora", MimeUtil::THEORA},
                 {"dts", MimeUtil::DTS}, {"mp2", MimeUtil::MP2},
                 {"mp4v.20.8", MimeUtil::MPEG4_SP}, {"mp4v.20.240", MimeUtil::MPEG4_ASP},
+                {"msmpeg4v1", MimeUtil::MSMPEG4v1}, {"msmpeg4v2", MimeUtil::MSMPEG4v2},
+                {"msmpeg4v3", MimeUtil::MSMPEG4v3}, {"wmv7", MimeUtil::WMV7},
+                {"wmv8", MimeUtil::WMV8}, {"wmv9", MimeUtil::WMV9},
+                {"vc-1", MimeUtil::VC1}, {"wma", MimeUtil::WMA},
+                {"vp6", MimeUtil::VP6}, {"mpeg1", MimeUtil::MPEG1}
           },
           base::KEEP_FIRST_OF_DUPES));
 
@@ -306,6 +311,9 @@ void MimeUtil::AddSupportedMediaFormats(
   CodecSet webm_codecs(webm_audio_codecs);
   webm_codecs.insert(webm_video_codecs.begin(), webm_video_codecs.end());
 
+  CodecSet mp1_codecs;
+  mp1_codecs.insert(MPEG1);
+
   const CodecSet mp3_codecs{MP3};
 
   CodecSet mp4_audio_codecs{FLAC, MP3, OPUS};
@@ -352,6 +360,29 @@ void MimeUtil::AddSupportedMediaFormats(
   CodecSet mp4_codecs(mp4_audio_codecs);
   mp4_codecs.insert(mp4_video_codecs.begin(), mp4_video_codecs.end());
 
+  CodecSet wma_codecs;
+  wma_codecs.insert(WMA);
+
+  CodecSet wmv_codecs(wma_codecs);
+  wmv_codecs.insert(MSMPEG4v1);
+  wmv_codecs.insert(MSMPEG4v2);
+  wmv_codecs.insert(MSMPEG4v3);
+  wmv_codecs.insert(MPEG4_SP);
+  wmv_codecs.insert(MPEG4_ASP);
+  wmv_codecs.insert(WMV7);
+  wmv_codecs.insert(WMV8);
+  wmv_codecs.insert(WMV9);
+  wmv_codecs.insert(VC1);
+
+  matroska_codecs.insert(MSMPEG4v1);
+  matroska_codecs.insert(MSMPEG4v2);
+  matroska_codecs.insert(MSMPEG4v3);
+
+  CodecSet flv_codecs;
+  flv_codecs.insert(VP6);
+  flv_codecs.insert(mp4_codecs.begin(), mp4_codecs.end());
+  flv_codecs.insert(MP3);
+
   const CodecSet implicit_codec;
   AddContainerWithCodecs("audio/wav", wav_codecs, false);
   AddContainerWithCodecs("audio/x-wav", wav_codecs, false);
@@ -375,6 +406,11 @@ void MimeUtil::AddSupportedMediaFormats(
 
   AddContainerWithCodecs("audio/x-matroska", matroska_audio_codecs, false);
   AddContainerWithCodecs("video/x-matroska", matroska_video_codecs, false);
+  AddContainerWithCodecs("video/x-ms-wmv", wmv_codecs, true);
+  AddContainerWithCodecs("video/x-ms-wma", wma_codecs, true);
+  AddContainerWithCodecs("video/x-ms-asf", wmv_codecs, true);
+  AddContainerWithCodecs("video/x-flv", flv_codecs, true);
+  AddContainerWithCodecs("video/mpeg", mp1_codecs, true);
 
 #if BUILDFLAG(USE_PROPRIETARY_CODECS)
   AddContainerWithCodecs("audio/aac", implicit_codec, true);  // AAC / ADTS.
@@ -1007,6 +1043,15 @@ bool MimeUtil::IsCodecProprietary(Codec
     case DTS:
     case MPEG4_SP:
     case MPEG4_ASP:
+    case MSMPEG4v1:
+    case MSMPEG4v2:
+    case MSMPEG4v3:
+    case WMV7:
+    case WMV8:
+    case WMV9:
+    case VC1:
+    case WMA:
+    case VP6:
       return true;
 
     case MP3:
@@ -1018,6 +1063,7 @@ bool MimeUtil::IsCodecProprietary(Codec
     case VP8:
     case VP9:
     case THEORA:
+    case MPEG1:
     case AV1:
       return false;
   }
diff -up chromium-69.0.3497.100/media/base/mime_util_internal.h.38~ chromium-69.0.3497.100/media/base/mime_util_internal.h
--- chromium-69.0.3497.100/media/base/mime_util_internal.h.38~	2018-10-02 22:33:23.202267666 +0200
+++ chromium-69.0.3497.100/media/base/mime_util_internal.h	2018-10-02 22:33:23.205267683 +0200
@@ -50,7 +50,17 @@ class MEDIA_EXPORT MimeUtil {
     MP2,
     MPEG4_SP,
     MPEG4_ASP,
-    LAST_CODEC = MPEG4_ASP
+    MSMPEG4v1,
+    MSMPEG4v2,
+    MSMPEG4v3,
+    WMV7,
+    WMV8,
+    WMV9,
+    VC1,
+    WMA,
+    VP6,
+    MPEG1,
+    LAST_CODEC = MPEG1
   };
 
   // Platform configuration structure.  Controls which codecs are supported at
diff -up chromium-69.0.3497.100/net/base/mime_util.cc.38~ chromium-69.0.3497.100/net/base/mime_util.cc
--- chromium-69.0.3497.100/net/base/mime_util.cc.38~	2018-10-02 22:33:23.203267672 +0200
+++ chromium-69.0.3497.100/net/base/mime_util.cc	2018-10-02 22:33:23.205267683 +0200
@@ -104,6 +104,10 @@ static const MimeInfo kPrimaryMappings[]
     {"video/x-matroska", "mkv"},
     {"audio/x-matroska", "mkv"},
     {"video/x-msvideo", "avi"},
+    {"video/x-ms-wmv", "wmv"},
+    {"audio/x-ms-wma", "wma"},
+    {"video/x-flv", "flv"},
+    {"video/mpeg", "mpg,mpeg"}
 };
 
 static const MimeInfo kSecondaryMappings[] = {
@@ -510,6 +514,7 @@ static const char* const kStandardVideoT
   "video/sd-video",
   "video/webm",
   "video/x-dv",
+  "video/x-flv",
   "video/x-m4v",
   "video/x-matroska",
   "video/x-mpeg",
