diff -up chromium-140.0.7339.80/chrome/browser/profiling_host/BUILD.gn.39~ chromium-140.0.7339.80/chrome/browser/profiling_host/BUILD.gn
--- chromium-140.0.7339.80/chrome/browser/profiling_host/BUILD.gn.39~	2025-08-29 20:50:09.000000000 +0200
+++ chromium-140.0.7339.80/chrome/browser/profiling_host/BUILD.gn	2025-09-06 19:38:57.174586287 +0200
@@ -25,7 +25,6 @@ static_library("profiling_host") {
     "//components/version_info",
     "//content/public/browser",
     "//content/public/common",
-    "//third_party/zlib",
 
     # Added to support the dependency on //chrome/browser/metrics
     # TODO(darin): We should instead have a way to express that dependency here.
