diff -up chromium-120.0.6099.71/build/linux/unbundle/replace_gn_files.py.20~ chromium-120.0.6099.71/build/linux/unbundle/replace_gn_files.py
--- chromium-120.0.6099.71/build/linux/unbundle/replace_gn_files.py.20~	2023-12-12 11:23:03.637514316 +0100
+++ chromium-120.0.6099.71/build/linux/unbundle/replace_gn_files.py	2023-12-12 11:26:02.253569761 +0100
@@ -79,6 +79,7 @@ REPLACEMENTS = {
   'vulkan_memory_allocator' : 'third_party/vulkan_memory_allocator/BUILD.gn',
   'woff2': 'third_party/woff2/BUILD.gn',
   'zlib': 'third_party/zlib/BUILD.gn',
+  'zstd': 'third_party/zstd/BUILD.gn',
 }
 
 
diff -up chromium-120.0.6099.71/build/linux/unbundle/zstd.gn.20~ chromium-120.0.6099.71/build/linux/unbundle/zstd.gn
--- chromium-120.0.6099.71/build/linux/unbundle/zstd.gn.20~	2023-12-12 11:23:03.637514316 +0100
+++ chromium-120.0.6099.71/build/linux/unbundle/zstd.gn	2023-12-12 11:23:03.637514316 +0100
@@ -0,0 +1,30 @@
+import("//build/config/linux/pkg_config.gni")
+import("//build/shim_headers.gni")
+
+pkg_config("system_zstd") {
+  packages = [ "libzstd" ]
+}
+
+shim_headers("zstd_shim") {
+  root_path = "src/lib"
+  headers = [
+    "zdict.h",
+    "zstd.h",
+    "zstd_errors.h",
+  ]
+}
+
+source_set("zstd") {
+  deps = [ ":zstd_shim" ]
+  public_configs = [ ":system_zstd" ]
+}
+
+source_set("compress") {
+  deps = [ ":zstd_shim" ]
+  public_configs = [ ":system_zstd" ]
+}
+
+source_set("decompress") {
+  deps = [ ":zstd_shim" ]
+  public_configs = [ ":system_zstd" ]
+}
