diff -up chromium-86.0.4238.0/build/config/compiler/BUILD.gn.32~ chromium-86.0.4238.0/build/config/compiler/BUILD.gn
--- chromium-86.0.4238.0/build/config/compiler/BUILD.gn.32~	2020-08-19 11:52:56.000000000 +0200
+++ chromium-86.0.4238.0/build/config/compiler/BUILD.gn	2020-08-21 02:48:31.501920496 +0200
@@ -395,14 +395,6 @@ config("compiler") {
         "-Wl,--as-needed",
       ]
     }
-    if (use_lld && !is_chromeos_device) {
-      # TODO(thakis): Fix dump_syms to not need this and then remove it,
-      # https://crbug.com/1105559
-      ldflags += [
-        "-Wl,-z,dead-reloc-in-nonalloc=*=0",
-        "-Wl,-z,dead-reloc-in-nonalloc=.debug_ranges=1",
-      ]
-    }
   }
 
   # Linux-specific compiler flags setup.
