diff -up chromium-69.0.3497.81/third_party/widevine/cdm/BUILD.gn.wvhack chromium-69.0.3497.81/third_party/widevine/cdm/BUILD.gn
--- chromium-69.0.3497.81/third_party/widevine/cdm/BUILD.gn.wvhack	2018-09-06 13:28:22.655272555 -0400
+++ chromium-69.0.3497.81/third_party/widevine/cdm/BUILD.gn	2018-09-06 13:28:22.655272555 -0400
@@ -11,7 +11,7 @@ import("//third_party/widevine/cdm/widev
 # Internal Cast builds set enable_widevine=true to bring in Widevine support.
 # TODO(xhwang): Support component updated CDM on other platforms and remove this
 # assert.
-assert(!enable_widevine || is_win || is_mac || is_chromecast,
+assert(!enable_widevine || is_win || is_mac || is_chromecast || is_linux,
        "Component updated CDM only supported on Windows and Mac for now.")
 
 widevine_arch = current_cpu
diff -up chromium-69.0.3497.81/third_party/widevine/cdm/stub/widevine_cdm_version.h.wvhack chromium-69.0.3497.81/third_party/widevine/cdm/stub/widevine_cdm_version.h
--- chromium-69.0.3497.81/third_party/widevine/cdm/stub/widevine_cdm_version.h.wvhack	2018-09-06 13:28:22.654272577 -0400
+++ chromium-69.0.3497.81/third_party/widevine/cdm/stub/widevine_cdm_version.h	2018-09-06 13:28:22.654272577 -0400
@@ -10,6 +10,7 @@
 
 #include "third_party/widevine/cdm/widevine_cdm_common.h"
 
+#define WIDEVINE_CDM_VERSION_STRING "unknown"
 #define WIDEVINE_CDM_AVAILABLE
 
 #endif  // WIDEVINE_CDM_VERSION_H_
