--- chromium-21.0.1171.0/third_party/WebKit/Source/WTF/wtf/RefPtr.h.orig	2012-06-15 20:48:49.000000000 +0000
+++ chromium-21.0.1171.0/third_party/WebKit/Source/WTF/wtf/RefPtr.h	2012-06-15 20:49:00.000000000 +0000
@@ -53,7 +53,7 @@ namespace WTF {
         RefPtr(HashTableDeletedValueType) : m_ptr(hashTableDeletedValue()) { }
         bool isHashTableDeletedValue() const { return m_ptr == hashTableDeletedValue(); }
 
-        ALWAYS_INLINE ~RefPtr() { derefIfNotNull(m_ptr); }
+        /*ALWAYS_INLINE*/ ~RefPtr() { derefIfNotNull(m_ptr); }
 
         T* get() const { return m_ptr; }
         
