diff -up chromium-81.0.4044.92/base/test/BUILD.gn.10~ chromium-81.0.4044.92/base/test/BUILD.gn
--- chromium-81.0.4044.92/base/test/BUILD.gn.10~	2020-04-12 15:29:43.843343651 +0200
+++ chromium-81.0.4044.92/base/test/BUILD.gn	2020-04-12 15:30:40.965979196 +0200
@@ -245,9 +245,6 @@ static_library("test_support") {
   if (is_linux) {
     public_deps += [ ":fontconfig_util_linux" ]
     data_deps = [ "//third_party/test_fonts" ]
-    if (current_toolchain == host_toolchain) {
-      data_deps += [ ":do_generate_fontconfig_caches" ]
-    }
   }
 
   if (is_ios) {
