diff -up chromium-135.0.7049.84/third_party/blink/renderer/build/scripts/gperf.py.omv~ chromium-135.0.7049.84/third_party/blink/renderer/build/scripts/gperf.py
--- chromium-135.0.7049.84/third_party/blink/renderer/build/scripts/gperf.py.omv~	2025-04-09 23:37:31.643606247 +0200
+++ chromium-135.0.7049.84/third_party/blink/renderer/build/scripts/gperf.py	2025-04-09 23:38:01.877820623 +0200
@@ -37,8 +37,9 @@ def generate_gperf(gperf_path, gperf_inp
         # -Wimplicit-fallthrough needs an explicit fallthrough statement,
         # so replace gperf's /*FALLTHROUGH*/ comment with the statement.
         # https://savannah.gnu.org/bugs/index.php?53029
-        gperf_output = gperf_output.replace('/*FALLTHROUGH*/',
-                                            '  [[fallthrough]];')
+# This is no longer needed with gperf 3.2
+#        gperf_output = gperf_output.replace('/*FALLTHROUGH*/',
+#                                            '  [[fallthrough]];')
         # -Wpointer-to-int-cast warns about casting pointers to smaller ints
         # Replace {(int)(long)&(foo), bar} with
         # {static_cast<int>(reinterpret_cast<uintptr_t>(&(foo)), bar}
