--- chromium-34.0.1847.134/build/gyp_chromium.orig	2014-05-07 13:02:57.752699020 +0100
+++ chromium-34.0.1847.134/build/gyp_chromium	2014-05-07 13:03:04.073699276 +0100
@@ -477,8 +477,6 @@
     args.append('--check')
 
   supplemental_includes = GetSupplementalFiles()
-  if not RunGN(supplemental_includes):
-    sys.exit(1)
   args.extend(
       ['-I' + i for i in additional_include_files(supplemental_includes, args)])
 
