diff -up chromium-84.0.4147.38/net/quic/platform/impl/quic_default_proof_providers_impl.cc.14~ chromium-84.0.4147.38/net/quic/platform/impl/quic_default_proof_providers_impl.cc
--- chromium-84.0.4147.38/net/quic/platform/impl/quic_default_proof_providers_impl.cc.14~	2020-06-06 20:24:02.716797104 +0200
+++ chromium-84.0.4147.38/net/quic/platform/impl/quic_default_proof_providers_impl.cc	2020-06-06 20:27:26.318274456 +0200
@@ -20,6 +20,7 @@
 #include "net/third_party/quiche/src/quic/platform/api/quic_flags.h"
 #include "net/third_party/quiche/src/quic/platform/api/quic_ptr_util.h"
 #include "net/third_party/quiche/src/quic/tools/simple_ticket_crypter.h"
+#include "net/quic/platform/impl/quic_flags_impl.cc"
 
 DEFINE_QUIC_COMMAND_LINE_FLAG(
     bool,
