diff --git a/media/base/mime_util_internal.cc b/media/base/mime_util_internal.cc
index ec03f4c..cc2db16 100644
--- a/media/base/mime_util_internal.cc
+++ b/media/base/mime_util_internal.cc
@@ -295,6 +295,7 @@ void MimeUtil::AddSupportedMediaFormats() {
   matroska_audio_codecs.insert(MPEG2_AAC);
   matroska_audio_codecs.insert(MPEG4_AAC);
   matroska_audio_codecs.insert(PCM);
+  matroska_audio_codecs.insert(FLAC);
 
   CodecSet matroska_video_codecs;
   matroska_video_codecs.insert(THEORA);
@@ -331,6 +332,7 @@ void MimeUtil::AddSupportedMediaFormats() {
 
   CodecSet mp4_audio_codecs(aac);
   mp4_audio_codecs.insert(MP3);
+  mp4_audio_codecs.insert(FLAC);
 #if BUILDFLAG(ENABLE_AC3_EAC3_AUDIO_DEMUXING)
   mp4_audio_codecs.insert(AC3);
   mp4_audio_codecs.insert(EAC3);
