diff -up chromium-112.0.5615.49/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h.11~ chromium-112.0.5615.49/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h
--- chromium-112.0.5615.49/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h.11~	2023-04-10 23:42:50.746505840 +0200
+++ chromium-112.0.5615.49/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h	2023-04-10 23:44:31.921353542 +0200
@@ -36,7 +36,7 @@
 #include <map>
 
 #include "common/using_std_string.h"
-#include "third_party/curl/curl.h"
+#include <curl/curl.h>
 
 namespace google_breakpad {
 
