diff -up chromium-107.0.5304.87/components/signin/internal/identity_manager/primary_account_manager.cc.omv~ chromium-107.0.5304.87/components/signin/internal/identity_manager/primary_account_manager.cc
--- chromium-107.0.5304.87/components/signin/internal/identity_manager/primary_account_manager.cc.omv~	2022-11-07 12:33:24.617596888 +0100
+++ chromium-107.0.5304.87/components/signin/internal/identity_manager/primary_account_manager.cc	2022-11-07 12:33:42.527918191 +0100
@@ -148,7 +148,11 @@ void PrimaryAccountManager::SetSyncPrima
   {
     std::string pref_account_id;
     bool consented_to_sync =
+#if 0
         client_->GetPrefs()->GetBoolean(prefs::kGoogleServicesConsentedToSync);
+#else
+        false;
+#endif
 
     DCHECK(pref_account_id.empty() || !consented_to_sync ||
            pref_account_id == account_info.account_id.ToString())
