diff -up chromium-79.0.3945.56/chrome/common/chrome_paths.cc.widevine-other-locations chromium-79.0.3945.56/chrome/common/chrome_paths.cc
--- chromium-79.0.3945.56/chrome/common/chrome_paths.cc.widevine-other-locations	2019-12-03 11:52:11.682295001 -0500
+++ chromium-79.0.3945.56/chrome/common/chrome_paths.cc	2019-12-03 12:00:08.654975939 -0500
@@ -381,6 +381,15 @@ bool PathProvider(int key, base::FilePat
 
 #if defined(OS_LINUX) && BUILDFLAG(BUNDLE_WIDEVINE_CDM)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      base::PathService::Get(base::DIR_HOME, &cur);
+      cur = cur.Append(FILE_PATH_LITERAL(".local/lib/libwidevinecdm.so"));
+      if (base::PathExists(cur)) {
+        break;
+      }
+      if (base::PathExists(base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/libwidevinecdm.so")))) {
+        cur = base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/libwidevinecdm.so"));
+        break;
+      }
       if (!GetComponentDirectory(&cur))
         return false;
 #if !defined(OS_CHROMEOS)
