diff -up chromium-78.0.3904.70/third_party/perfetto/gn/BUILD.gn.unbundle-zlib chromium-78.0.3904.70/third_party/perfetto/gn/BUILD.gn
--- chromium-78.0.3904.70/third_party/perfetto/gn/BUILD.gn.unbundle-zlib	2019-10-23 09:25:44.419884187 -0400
+++ chromium-78.0.3904.70/third_party/perfetto/gn/BUILD.gn	2019-10-23 09:29:53.553442745 -0400
@@ -276,7 +276,7 @@ if (enable_perfetto_trace_processor || e
         "//buildtools:zlib",
       ]
     } else {
-      public_configs = [ "//third_party/zlib:zlib_config" ]
+      public_configs = [ "//third_party/zlib:system_zlib" ]
       public_deps = [
         "//third_party/zlib",
       ]
