diff -up chromium-89.0.4389.47/base/BUILD.gn.23~ chromium-89.0.4389.47/base/BUILD.gn
--- chromium-89.0.4389.47/base/BUILD.gn.23~	2021-02-12 13:25:45.387277309 +0100
+++ chromium-89.0.4389.47/base/BUILD.gn	2021-02-12 13:27:09.044749663 +0100
@@ -3257,7 +3257,7 @@ if (build_base_unittests) {
       ]
     }
 
-    if (icu_use_data_file) {
+    if (false) {
       if (is_android) {
         deps += [ "//third_party/icu:icu_extra_assets" ]
       } else {
diff -up chromium-89.0.4389.47/third_party/blink/renderer/platform/text/character_property_data.h.23~ chromium-89.0.4389.47/third_party/blink/renderer/platform/text/character_property_data.h
--- chromium-89.0.4389.47/third_party/blink/renderer/platform/text/character_property_data.h.23~	2021-02-11 00:04:08.000000000 +0100
+++ chromium-89.0.4389.47/third_party/blink/renderer/platform/text/character_property_data.h	2021-02-12 13:25:45.386277291 +0100
@@ -161,10 +161,12 @@ static const UChar32 kIsHangulRanges[] =
     0xD7B0, 0xD7FF,
     // Halfwidth Hangul Jamo
     // https://www.unicode.org/charts/nameslist/c_FF00.html
-    0xFFA0, 0xFFDC,
+    0xFFA0, 0xFFDB,
 };
 
-static const UChar32 kIsHangulArray[] = {};
+static const UChar32 kIsHangulArray[] = {
+  0xFFDC,
+};
 
 // Freezed trie tree, see character_property_data_generator.cc.
 extern const int32_t kSerializedCharacterDataSize;
