--- chromium-65.0.3325.146/media/base/decoder_buffer.h.omv~	2018-03-09 00:40:33.076776781 +0100
+++ chromium-65.0.3325.146/media/base/decoder_buffer.h	2018-03-09 00:46:47.614847601 +0100
@@ -8,6 +8,8 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <libavcodec/avcodec.h>
+
 #include <memory>
 #include <string>
 #include <utility>
@@ -37,7 +39,7 @@ class MEDIA_EXPORT DecoderBuffer
     : public base::RefCountedThreadSafe<DecoderBuffer> {
  public:
   enum {
-    kPaddingSize = 64,
+    kPaddingSize = AV_INPUT_BUFFER_PADDING_SIZE > 64 ? AV_INPUT_BUFFER_PADDING_SIZE : 64,
 #if defined(ARCH_CPU_ARM_FAMILY)
     kAlignmentSize = 16
 #else
