diff -up chromium-120.0.6099.71/build/config/compiler/BUILD.gn.omv~ chromium-120.0.6099.71/build/config/compiler/BUILD.gn
--- chromium-120.0.6099.71/build/config/compiler/BUILD.gn.omv~	2023-12-12 12:37:20.679876703 +0100
+++ chromium-120.0.6099.71/build/config/compiler/BUILD.gn	2023-12-12 12:38:15.149217007 +0100
@@ -619,20 +619,20 @@ config("compiler") {
     # TODO(crbug.com/1488374): This causes binary size growth and potentially
     # other problems.
     # TODO(crbug.com/1491036): This isn't supported by Cronet's mainline llvm version.
-    if (default_toolchain != "//build/toolchain/cros:target" &&
-        !llvm_android_mainline) {
-      cflags += [
-        "-mllvm",
-        "-split-threshold-for-reg-with-hint=0",
-      ]
-      if (use_thin_lto && is_a_target_toolchain) {
-        if (is_win) {
-          ldflags += [ "-mllvm:-split-threshold-for-reg-with-hint=0" ]
-        } else {
-          ldflags += [ "-Wl,-mllvm,-split-threshold-for-reg-with-hint=0" ]
-        }
-      }
-    }
+    #if (default_toolchain != "//build/toolchain/cros:target" &&
+    #    !llvm_android_mainline) {
+    #  cflags += [
+    #    "-mllvm",
+    #    "-split-threshold-for-reg-with-hint=0",
+    #  ]
+    #  if (use_thin_lto && is_a_target_toolchain) {
+    #    if (is_win) {
+    #      ldflags += [ "-mllvm:-split-threshold-for-reg-with-hint=0" ]
+    #    } else {
+    #      ldflags += [ "-Wl,-mllvm,-split-threshold-for-reg-with-hint=0" ]
+    #    }
+    #  }
+    #}
 
     # TODO(crbug.com/1235145): Investigate why/if this should be needed.
     if (is_win) {
