diff -up chromium-79.0.3945.56/content/common/user_agent.cc.fedora-user-agent chromium-79.0.3945.56/content/common/user_agent.cc
--- chromium-79.0.3945.56/content/common/user_agent.cc.fedora-user-agent	2019-12-03 14:16:13.076763501 -0500
+++ chromium-79.0.3945.56/content/common/user_agent.cc	2019-12-03 14:21:57.427351621 -0500
@@ -35,7 +35,7 @@ std::string GetUserAgentPlatform() {
 #elif defined(OS_MACOSX)
   return "Macintosh; ";
 #elif defined(USE_X11) || defined(USE_OZONE)
-  return "X11; ";  // strange, but that's what Firefox uses
+  return "X11; Fedora; ";  // strange, but that's what Firefox uses
 #elif defined(OS_ANDROID)
   return "Linux; ";
 #elif defined(OS_FUCHSIA)
