diff -up chromium-119.0.6045.105/services/device/public/cpp/generic_sensor/sensor_reading.h.24~ chromium-119.0.6045.105/services/device/public/cpp/generic_sensor/sensor_reading.h
--- chromium-119.0.6045.105/services/device/public/cpp/generic_sensor/sensor_reading.h.24~	2023-11-01 21:10:09.242254951 +0100
+++ chromium-119.0.6045.105/services/device/public/cpp/generic_sensor/sensor_reading.h	2023-11-01 21:11:31.751105967 +0100
@@ -9,6 +9,7 @@
 #include <stdint.h>
 
 #include <type_traits>
+#include <cstdint>
 
 namespace device {
 
