diff -up chromium-130.0.6723.58/media/video/av1_video_encoder.cc.51~ chromium-130.0.6723.58/media/video/av1_video_encoder.cc
--- chromium-130.0.6723.58/media/video/av1_video_encoder.cc.51~	2024-10-17 21:44:24.583132769 +0200
+++ chromium-130.0.6723.58/media/video/av1_video_encoder.cc	2024-10-17 21:46:25.874116347 +0200
@@ -546,7 +546,9 @@ void Av1VideoEncoder::Encode(scoped_refp
     int qp = static_cast<int>(std::lround(encode_options.quantizer.value()));
     qp = std::clamp(qp, static_cast<int>(config_.rc_min_quantizer),
                     static_cast<int>(config_.rc_max_quantizer));
+#ifdef AV1E_SET_QUANTIZER_ONE_PASS
     aom_codec_control(codec_.get(), AV1E_SET_QUANTIZER_ONE_PASS, qp);
+#endif
   }
 
   if (options_.manual_reference_buffer_control) {
