--- chromium-62.0.3202.89/media/base/mime_util_internal.cc.0112~	2017-11-12 16:11:06.596851305 +0100
+++ chromium-62.0.3202.89/media/base/mime_util_internal.cc	2017-11-12 16:13:08.837069343 +0100
@@ -75,6 +75,11 @@ const base::flat_map<std::string, MimeUt
         {"vp8.0", MimeUtil::VP8}, {"theora", MimeUtil::THEORA},
 	{"dts", MimeUtil::DTS}, {"mp2", MimeUtil::MP2},
         {"mp4v.20.8", MimeUtil::MPEG4_SP}, {"mp4v.20.240", MimeUtil::MPEG4_ASP},
+        {"msmpeg4v1", MimeUtil::MSMPEG4v1}, {"msmpeg4v2", MimeUtil::MSMPEG4v2},
+        {"msmpeg4v3", MimeUtil::MSMPEG4v3}, {"wmv7", MimeUtil::WMV7},
+        {"wmv8", MimeUtil::WMV8}, {"wmv9", MimeUtil::WMV9},
+        {"vc-1", MimeUtil::VC1}, {"wma", MimeUtil::WMA},
+        {"vp6", MimeUtil::VP6}, {"mpeg1", MimeUtil::MPEG1},
       },
       base::KEEP_FIRST_OF_DUPES);
 
@@ -300,6 +305,9 @@ void MimeUtil::AddSupportedMediaFormats(
   const CodecSet mp3_codecs{MP3};
   const CodecSet aac{MPEG2_AAC, MPEG4_AAC};
 
+  CodecSet mp1_codecs;
+  mp1_codecs.emplace(MPEG1);
+
   CodecSet avc_and_aac(aac);
   avc_and_aac.emplace(H264);
 
@@ -328,6 +336,29 @@ void MimeUtil::AddSupportedMediaFormats(
   mp4_codecs.insert(mp4_video_codecs.begin(), mp4_video_codecs.end());
 #endif  // BUILDFLAG(USE_PROPRIETARY_CODECS)
 
+  CodecSet wma_codecs;
+  wma_codecs.insert(WMA);
+  
+  CodecSet wmv_codecs(wma_codecs);
+  wmv_codecs.insert(MSMPEG4v1);
+  wmv_codecs.insert(MSMPEG4v2);
+  wmv_codecs.insert(MSMPEG4v3);
+  wmv_codecs.insert(MPEG4_SP);
+  wmv_codecs.insert(MPEG4_ASP);
+  wmv_codecs.insert(WMV7);
+  wmv_codecs.insert(WMV8);
+  wmv_codecs.insert(WMV9);
+  wmv_codecs.insert(VC1);
+  
+  matroska_codecs.insert(MSMPEG4v1);
+  matroska_codecs.insert(MSMPEG4v2);
+  matroska_codecs.insert(MSMPEG4v3);
+  
+  CodecSet flv_codecs;
+  flv_codecs.insert(VP6);
+  flv_codecs.insert(mp4_codecs.begin(), mp4_codecs.end());
+  flv_codecs.insert(MP3);
+
   const CodecSet implicit_codec;
   AddContainerWithCodecs("audio/wav", wav_codecs, false);
   AddContainerWithCodecs("audio/x-wav", wav_codecs, false);
@@ -345,6 +376,11 @@ void MimeUtil::AddSupportedMediaFormats(
 
   AddContainerWithCodecs("audio/x-matroska", matroska_audio_codecs, false);
   AddContainerWithCodecs("video/x-matroska", matroska_codecs, false);
+  AddContainerWithCodecs("video/x-ms-wmv", wmv_codecs, true);
+  AddContainerWithCodecs("audio/x-ms-wma", wma_codecs, true);
+  AddContainerWithCodecs("video/x-ms-asf", wmv_codecs, true);
+  AddContainerWithCodecs("video/x-flv", flv_codecs, true);
+  AddContainerWithCodecs("video/mpeg", mp1_codecs, true);
 
 #if BUILDFLAG(USE_PROPRIETARY_CODECS)
   AddContainerWithCodecs("audio/mpeg", mp3_codecs, true);  // Allow "mp3".
@@ -965,6 +1001,15 @@ bool MimeUtil::IsCodecProprietary(Codec
     case DTS:
     case MPEG4_SP:
     case MPEG4_ASP:
+    case MSMPEG4v1:
+    case MSMPEG4v2:
+    case MSMPEG4v3:
+    case WMV7:
+    case WMV8:
+    case WMV9:
+    case VC1:
+    case WMA:
+    case VP6:
       return true;
 
     case PCM:
@@ -975,6 +1020,7 @@ bool MimeUtil::IsCodecProprietary(Codec
     case VP8:
     case VP9:
     case THEORA:
+    case MPEG1:
       return false;
   }
 
--- chromium-62.0.3202.89/media/base/mime_util_internal.h.0112~	2017-11-12 16:11:06.597851292 +0100
+++ chromium-62.0.3202.89/media/base/mime_util_internal.h	2017-11-12 16:11:06.598851279 +0100
@@ -48,7 +48,17 @@ class MEDIA_EXPORT MimeUtil {
     MP2,
     MPEG4_SP,
     MPEG4_ASP,
-    LAST_CODEC = MPEG4_ASP
+    MSMPEG4v1,
+    MSMPEG4v2,
+    MSMPEG4v3,
+    WMV7,
+    WMV8,
+    WMV9,
+    VC1,
+    WMA,
+    VP6,
+    MPEG1,
+    LAST_CODEC = MPEG1
   };
 
   // Platform configuration structure.  Controls which codecs are supported at
--- chromium-62.0.3202.89/net/base/mime_util.cc.0112~	2017-11-12 16:11:06.597851292 +0100
+++ chromium-62.0.3202.89/net/base/mime_util.cc	2017-11-12 16:11:06.598851279 +0100
@@ -512,6 +512,7 @@ static const char* const kStandardVideoT
   "video/sd-video",
   "video/webm",
   "video/x-dv",
+  "video/x-flv",
   "video/x-m4v",
   "video/x-matroska",
   "video/x-mpeg",
