diff -up chromium-92.0.4515.131/base/BUILD.gn.21~ chromium-92.0.4515.131/base/BUILD.gn
--- chromium-92.0.4515.131/base/BUILD.gn.21~	2021-08-14 04:50:54.077202236 +0200
+++ chromium-92.0.4515.131/base/BUILD.gn	2021-08-14 04:52:08.212268230 +0200
@@ -3469,7 +3469,7 @@ test("base_unittests") {
     ]
   }
 
-  if (icu_use_data_file) {
+  if (false) {
     if (is_android) {
       deps += [ "//third_party/icu:icu_extra_assets" ]
     } else {
diff -up chromium-92.0.4515.131/third_party/blink/renderer/platform/text/character_property_data.h.21~ chromium-92.0.4515.131/third_party/blink/renderer/platform/text/character_property_data.h
--- chromium-92.0.4515.131/third_party/blink/renderer/platform/text/character_property_data.h.21~	2021-08-03 04:50:29.000000000 +0200
+++ chromium-92.0.4515.131/third_party/blink/renderer/platform/text/character_property_data.h	2021-08-14 04:50:54.077202236 +0200
@@ -161,10 +161,12 @@ static const UChar32 kIsHangulRanges[] =
     0xD7B0, 0xD7FF,
     // Halfwidth Hangul Jamo
     // https://www.unicode.org/charts/nameslist/c_FF00.html
-    0xFFA0, 0xFFDC,
+    0xFFA0, 0xFFDB,
 };
 
-static const UChar32 kIsHangulArray[] = {};
+static const UChar32 kIsHangulArray[] = {
+  0xFFDC,
+};
 
 // Freezed trie tree, see character_property_data_generator.cc.
 extern const int32_t kSerializedCharacterDataSize;
