--- chromium-60.0.3112.20.old/components/version_ui_strings.grdp	2017-06-08 16:55:00.495103433 +0200
+++ chromium-60.0.3112.20/components/version_ui_strings.grdp	2017-06-08 16:55:36.027102220 +0200
@@ -7,7 +7,7 @@
     Official Build
   </message>
   <message name="IDS_VERSION_UI_UNOFFICIAL" desc="unofficial build on the about:version page">
-    Developer Build
+    OpenMandriva Build
   </message>
   <message name="IDS_VERSION_UI_32BIT" desc="32-bit on the chrome://version page">
     (32-bit)
