diff -up chromium-128.0.6613.113/base/containers/span.h.omv~ chromium-128.0.6613.113/base/containers/span.h
--- chromium-128.0.6613.113/base/containers/span.h.omv~	2024-08-31 01:25:25.360511913 +0200
+++ chromium-128.0.6613.113/base/containers/span.h	2024-08-31 01:25:33.860578616 +0200
@@ -7,6 +7,7 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#include <cstring>
 
 #include <algorithm>
 #include <array>
diff -up chromium-128.0.6613.113/build/rust/std/remap_alloc.cc.omv~ chromium-128.0.6613.113/build/rust/std/remap_alloc.cc
--- chromium-128.0.6613.113/build/rust/std/remap_alloc.cc.omv~	2024-08-31 01:08:41.361042351 +0200
+++ chromium-128.0.6613.113/build/rust/std/remap_alloc.cc	2024-08-31 01:08:46.331072780 +0200
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #include <cstddef>
+#include <cstring>
 
 #include "build/build_config.h"
 #include "build/rust/std/alias.h"
diff -up chromium-128.0.6613.113/components/enterprise/connectors/common.h.omv~ chromium-128.0.6613.113/components/enterprise/connectors/common.h
--- chromium-128.0.6613.113/components/enterprise/connectors/common.h.omv~	2024-08-31 17:00:51.361510751 +0200
+++ chromium-128.0.6613.113/components/enterprise/connectors/common.h	2024-08-31 17:00:56.941570933 +0200
@@ -10,6 +10,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <optional>
 
 #include "base/files/file_path.h"
 #include "base/supports_user_data.h"
diff -up chromium-128.0.6613.113/components/performance_manager/persistence/site_data/site_data_writer.h.omv~ chromium-128.0.6613.113/components/performance_manager/persistence/site_data/site_data_writer.h
