diff -up chromium-61.0.3163.79/tools/gn/bootstrap/bootstrap.py.fixit chromium-61.0.3163.79/tools/gn/bootstrap/bootstrap.py
--- chromium-61.0.3163.79/tools/gn/bootstrap/bootstrap.py.fixit	2017-09-06 15:30:19.199783450 -0400
+++ chromium-61.0.3163.79/tools/gn/bootstrap/bootstrap.py	2017-09-06 15:31:17.781654042 -0400
@@ -449,6 +449,7 @@ def write_gn_ninja(path, root_gen_dir, o
       'base/metrics/histogram_base.cc',
       'base/metrics/histogram_functions.cc',
       'base/metrics/histogram_samples.cc',
+      'base/metrics/histogram_snapshot_manager.cc',
       'base/metrics/metrics_hashes.cc',
       'base/metrics/persistent_histogram_allocator.cc',
       'base/metrics/persistent_memory_allocator.cc',
@@ -534,7 +535,7 @@ def write_gn_ninja(path, root_gen_dir, o
       'base/trace_event/heap_profiler_allocation_context_tracker.cc',
       'base/trace_event/heap_profiler_allocation_register.cc',
       'base/trace_event/heap_profiler_event_filter.cc',
-      'base/trace_event/heap_profiler_event_writer.cc',
+      'base/trace_event/heap_profiler_heap_dump_writer.cc',
       'base/trace_event/heap_profiler_serialization_state.cc',
       'base/trace_event/heap_profiler_stack_frame_deduplicator.cc',
       'base/trace_event/heap_profiler_type_name_deduplicator.cc',
