--- chromium-35.0.1916.153/build/linux/system.gyp.0013~	2014-06-13 17:45:54.284344191 +0200
+++ chromium-35.0.1916.153/build/linux/system.gyp	2014-06-13 17:51:00.946641566 +0200
@@ -1011,5 +1011,41 @@
         }],
       ],
     },
+    {
+      'target_name': 'icu',
+      'type': 'none',
+      'conditions': [
+        ['_toolset=="target"', {
+          'direct_dependent_settings': {
+            'cflags': [
+              '-Wno-unused-function',  
+              # TODO(port): fix ICU to not depend on this flag.  
+              '-fno-strict-aliasing',
+            ],
+          },
+          'link_settings': {
+            'ldflags': [
+              '',
+            ],
+            'libraries': [
+              '-licui18n',
+            ],
+          },
+      }]]   
+    },   
+    {
+      'target_name': 'glew',
+      'type': 'none',
+      'conditions': [
+        ['_toolset=="target"', {
+          'link_settings': {
+            'libraries': [
+              '-lGLEW',
+              '-lGL',
+              '-lX11',
+            ],
+          },
+      }]]
+    },
   ],
 }
