diff -up chromium-93.0.4577.42/media/base/audio_codecs.cc.27~ chromium-93.0.4577.42/media/base/audio_codecs.cc
--- chromium-93.0.4577.42/media/base/audio_codecs.cc.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/base/audio_codecs.cc	2021-08-15 05:51:57.694300788 +0200
@@ -45,6 +45,10 @@ std::string GetCodecName(AudioCodec code
       return "ac3";
     case kCodecMpegHAudio:
       return "mpeg-h-audio";
+    case kCodecMP2:
+      return "mp2";
+    case kCodecDTS:
+      return "dts";
   }
 }
 
@@ -80,6 +84,10 @@ AudioCodec StringToAudioCodec(const std:
     return kCodecVorbis;
   if (base::StartsWith(codec_id, "mp4a.40.", base::CompareCase::SENSITIVE))
     return kCodecAAC;
+  if (codec_id == "mp2")
+    return kCodecMP2;
+  if (codec_id == "dts")
+    return kCodecDTS;
   return kUnknownAudioCodec;
 }
 
diff -up chromium-93.0.4577.42/media/base/audio_codecs.h.27~ chromium-93.0.4577.42/media/base/audio_codecs.h
--- chromium-93.0.4577.42/media/base/audio_codecs.h.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/base/audio_codecs.h	2021-08-15 05:51:57.696300776 +0200
@@ -33,13 +33,15 @@ enum AudioCodec {
   kCodecALAC = 15,
   kCodecAC3 = 16,
   kCodecMpegHAudio = 17,
+  kCodecMP2 = 18,
+  kCodecDTS = 19,
   // DO NOT ADD RANDOM AUDIO CODECS!
   //
   // The only acceptable time to add a new codec is if there is production code
   // that uses said codec in the same CL.
 
   // Must always be equal to the largest entry ever logged.
-  kAudioCodecMax = kCodecMpegHAudio,
+  kAudioCodecMax = kCodecDTS,
 };
 
 enum class AudioCodecProfile {
diff -up chromium-93.0.4577.42/media/base/mime_util_internal.cc.27~ chromium-93.0.4577.42/media/base/mime_util_internal.cc
--- chromium-93.0.4577.42/media/base/mime_util_internal.cc.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/base/mime_util_internal.cc	2021-08-15 05:51:57.696300776 +0200
@@ -91,6 +91,10 @@ const StringToCodecMap& GetStringToCodec
       {"vp8", MimeUtil::VP8},
       {"vp8.0", MimeUtil::VP8},
       {"theora", MimeUtil::THEORA},
+      {"dts", MimeUtil::DTS},
+      {"mp2", MimeUtil::MP2},
+      {"mp4v.20.8", MimeUtil::MPEG4_SP},
+      {"mp4v.20.240", MimeUtil::MPEG4_ASP},
   });
 
   return *kStringToCodecMap;
@@ -187,6 +191,10 @@ AudioCodec MimeUtilToAudioCodec(MimeUtil
       return kCodecOpus;
     case MimeUtil::FLAC:
       return kCodecFLAC;
+    case MimeUtil::DTS:
+      return kCodecDTS;
+    case MimeUtil::MP2:
+      return kCodecMP2;
     default:
       break;
   }
@@ -209,6 +217,9 @@ VideoCodec MimeUtilToVideoCodec(MimeUtil
       return kCodecTheora;
     case MimeUtil::DOLBY_VISION:
       return kCodecDolbyVision;
+    case MimeUtil::MPEG4_SP:
+    case MimeUtil::MPEG4_ASP:
+      return kCodecMPEG4;
     default:
       break;
   }
@@ -298,6 +309,33 @@ void MimeUtil::AddSupportedMediaFormats(
   CodecSet ogg_codecs(ogg_audio_codecs);
   ogg_codecs.insert(ogg_video_codecs.begin(), ogg_video_codecs.end());
 
+  CodecSet matroska_audio_codecs;
+  matroska_audio_codecs.emplace(OPUS);
+  matroska_audio_codecs.emplace(VORBIS);
+  matroska_audio_codecs.emplace(AC3);
+  matroska_audio_codecs.emplace(EAC3);
+  matroska_audio_codecs.emplace(DTS);
+  matroska_audio_codecs.emplace(MP3);
+  matroska_audio_codecs.emplace(MP2);
+  matroska_audio_codecs.emplace(MPEG2_AAC);
+  matroska_audio_codecs.emplace(MPEG4_AAC);
+  matroska_audio_codecs.emplace(PCM);
+  matroska_audio_codecs.emplace(FLAC);
+
+  CodecSet matroska_video_codecs;
+  matroska_video_codecs.emplace(THEORA);
+  matroska_video_codecs.emplace(VP8);
+  matroska_video_codecs.emplace(VP9);
+  matroska_video_codecs.emplace(H264);
+  matroska_video_codecs.emplace(MPEG4_SP);
+  matroska_video_codecs.emplace(MPEG4_ASP);
+#if BUILDFLAG(ENABLE_AV1_DECODER)
+  matroska_video_codecs.emplace(AV1);
+#endif
+  matroska_video_codecs.emplace(HEVC);
+  CodecSet matroska_codecs(matroska_audio_codecs);
+  matroska_codecs.insert(matroska_video_codecs.begin(), matroska_video_codecs.end());
+
   const CodecSet webm_audio_codecs{OPUS, VORBIS};
   CodecSet webm_video_codecs{VP8, VP9};
 #if BUILDFLAG(ENABLE_AV1_DECODER)
@@ -307,9 +345,9 @@ void MimeUtil::AddSupportedMediaFormats(
   CodecSet webm_codecs(webm_audio_codecs);
   webm_codecs.insert(webm_video_codecs.begin(), webm_video_codecs.end());
 
-  const CodecSet mp3_codecs{MP3};
+  const CodecSet mp3_codecs{MP3, MP2};
 
-  CodecSet mp4_audio_codecs{FLAC, MP3, OPUS};
+  CodecSet mp4_audio_codecs{FLAC, MP3, OPUS, MP2, DTS, AC3, EAC3};
 
   // Only VP9 with valid codec string vp09.xx.xx.xx.xx.xx.xx.xx is supported.
   // See ParseVp9CodecID for details.
@@ -369,6 +407,10 @@ void MimeUtil::AddSupportedMediaFormats(
   DCHECK(!mp4_video_codecs.empty());
   AddContainerWithCodecs("video/mp4", mp4_codecs);
 
+  AddContainerWithCodecs("audio/x-matroska", matroska_audio_codecs);
+  AddContainerWithCodecs("video/x-matroska", matroska_codecs);
+  AddContainerWithCodecs("video/x-msvideo", matroska_codecs);
+
 #if BUILDFLAG(USE_PROPRIETARY_CODECS)
   AddContainerWithCodecs("audio/aac", implicit_codec);  // AAC / ADTS.
   // These strings are supported for backwards compatibility only and thus only
diff -up chromium-93.0.4577.42/media/base/mime_util_internal.h.27~ chromium-93.0.4577.42/media/base/mime_util_internal.h
--- chromium-93.0.4577.42/media/base/mime_util_internal.h.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/base/mime_util_internal.h	2021-08-15 05:51:57.696300776 +0200
@@ -47,7 +47,11 @@ class MEDIA_EXPORT MimeUtil {
     DOLBY_VISION,
     AV1,
     MPEG_H_AUDIO,
-    LAST_CODEC = MPEG_H_AUDIO
+    DTS,
+    MP2,
+    MPEG4_SP,
+    MPEG4_ASP,
+    LAST_CODEC = MPEG4_ASP
   };
 
   // Platform configuration structure.  Controls which codecs are supported at
diff -up chromium-93.0.4577.42/media/base/supported_types.cc.27~ chromium-93.0.4577.42/media/base/supported_types.cc
--- chromium-93.0.4577.42/media/base/supported_types.cc.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/base/supported_types.cc	2021-08-15 05:53:46.672645263 +0200
@@ -209,6 +209,7 @@ bool IsAudioCodecProprietary(AudioCodec
     case kCodecAAC:
     case kCodecAC3:
     case kCodecEAC3:
+    case kCodecDTS:
     case kCodecAMR_NB:
     case kCodecAMR_WB:
     case kCodecGSM_MS:
@@ -217,6 +218,7 @@ bool IsAudioCodecProprietary(AudioCodec
       return true;
 
     case kCodecFLAC:
+    case kCodecMP2:
     case kCodecMP3:
     case kCodecOpus:
     case kCodecVorbis:
@@ -254,6 +256,7 @@ bool IsDefaultSupportedAudioType(const A
 #endif
 
     case kCodecFLAC:
+    case kCodecMP2:
     case kCodecMP3:
     case kCodecOpus:
     case kCodecPCM:
@@ -262,6 +265,10 @@ bool IsDefaultSupportedAudioType(const A
     case kCodecPCM_S24BE:
     case kCodecPCM_ALAW:
     case kCodecVorbis:
+    case media::kCodecEAC3:
+    case media::kCodecALAC:
+    case media::kCodecAC3:
+    case media::kCodecDTS:
       return true;
 
     case kCodecAMR_NB:
@@ -273,9 +280,6 @@ bool IsDefaultSupportedAudioType(const A
       return false;
 #endif
 
-    case kCodecEAC3:
-    case kCodecALAC:
-    case kCodecAC3:
     case kCodecMpegHAudio:
     case kUnknownAudioCodec:
       return false;
@@ -340,6 +344,10 @@ bool IsDefaultSupportedVideoType(const V
     case kCodecH264:
     case kCodecVP8:
     case kCodecTheora:
+    case kCodecVC1:
+    case kCodecMPEG2:
+    case kCodecDolbyVision:
+    case kCodecMPEG4:
       return true;
 
     case kCodecHEVC:
@@ -350,17 +358,7 @@ bool IsDefaultSupportedVideoType(const V
       return false;
 #endif  // BUILDFLAG(ENABLE_PLATFORM_ENCRYPTED_HEVC)
     case kUnknownVideoCodec:
-    case kCodecVC1:
-    case kCodecMPEG2:
-    case kCodecDolbyVision:
-      return false;
-
-    case kCodecMPEG4:
-#if BUILDFLAG(IS_CHROMEOS_ASH)
-      return true;
-#else
       return false;
-#endif
   }
 
   NOTREACHED();
diff -up chromium-93.0.4577.42/media/ffmpeg/ffmpeg_common.cc.27~ chromium-93.0.4577.42/media/ffmpeg/ffmpeg_common.cc
--- chromium-93.0.4577.42/media/ffmpeg/ffmpeg_common.cc.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/ffmpeg/ffmpeg_common.cc	2021-08-15 05:51:57.696300776 +0200
@@ -125,6 +125,10 @@ AudioCodec CodecIDToAudioCodec(AVCodecID
     case AV_CODEC_ID_MPEGH_3D_AUDIO:
       return kCodecMpegHAudio;
 #endif
+    case AV_CODEC_ID_MP2:
+      return kCodecMP2;
+    case AV_CODEC_ID_DTS:
+      return kCodecDTS;
     default:
       DVLOG(1) << "Unknown audio CodecID: " << codec_id;
   }
@@ -180,6 +184,14 @@ AVCodecID AudioCodecToCodecID(AudioCodec
     case kCodecMpegHAudio:
       return AV_CODEC_ID_MPEGH_3D_AUDIO;
 #endif
+    case kCodecAC3:
+      return AV_CODEC_ID_AC3;
+    case kCodecEAC3:
+      return AV_CODEC_ID_EAC3;
+    case kCodecMP2:
+      return AV_CODEC_ID_MP2;
+    case kCodecDTS:
+      return AV_CODEC_ID_DTS;
     default:
       DVLOG(1) << "Unknown AudioCodec: " << audio_codec;
   }
@@ -348,6 +360,7 @@ bool AVCodecContextToAudioDecoderConfig(
                                                codec_context->channels);
 
   int sample_rate = codec_context->sample_rate;
+#if 0
   switch (codec) {
     // For AC3/EAC3 we enable only demuxing, but not decoding, so FFmpeg does
     // not fill |sample_fmt|.
@@ -371,6 +384,7 @@ bool AVCodecContextToAudioDecoderConfig(
     default:
       break;
   }
+#endif
 
   base::TimeDelta seek_preroll;
   if (codec_context->seek_preroll > 0) {
diff -up chromium-93.0.4577.42/media/filters/ffmpeg_video_decoder.cc.27~ chromium-93.0.4577.42/media/filters/ffmpeg_video_decoder.cc
--- chromium-93.0.4577.42/media/filters/ffmpeg_video_decoder.cc.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/filters/ffmpeg_video_decoder.cc	2021-08-15 05:51:57.696300776 +0200
@@ -40,7 +40,6 @@ static int GetFFmpegVideoDecoderThreadCo
     case kUnknownVideoCodec:
     case kCodecVC1:
     case kCodecMPEG2:
-    case kCodecHEVC:
     case kCodecVP9:
     case kCodecAV1:
     case kCodecDolbyVision:
@@ -54,6 +53,7 @@ static int GetFFmpegVideoDecoderThreadCo
 
     case kCodecH264:
     case kCodecVP8:
+    case kCodecHEVC:
       // Normalize to three threads for 1080p content, then scale linearly
       // with number of pixels.
       // Examples:
diff -up chromium-93.0.4577.42/media/remoting/proto_enum_utils.cc.27~ chromium-93.0.4577.42/media/remoting/proto_enum_utils.cc
--- chromium-93.0.4577.42/media/remoting/proto_enum_utils.cc.27~	2021-08-12 21:10:18.000000000 +0200
+++ chromium-93.0.4577.42/media/remoting/proto_enum_utils.cc	2021-08-15 05:51:57.697300770 +0200
@@ -34,6 +34,8 @@ absl::optional<AudioCodec> ToMediaAudioC
     CASE_RETURN_OTHER(kCodecALAC);
     CASE_RETURN_OTHER(kCodecAC3);
     CASE_RETURN_OTHER(kCodecMpegHAudio);
+    CASE_RETURN_OTHER(kCodecMP2);
+    CASE_RETURN_OTHER(kCodecDTS);
   }
   return absl::nullopt;  // Not a 'default' to ensure compile-time checks.
 }
@@ -61,6 +63,8 @@ ToProtoAudioDecoderConfigCodec(AudioCode
     CASE_RETURN_OTHER(kCodecALAC);
     CASE_RETURN_OTHER(kCodecAC3);
     CASE_RETURN_OTHER(kCodecMpegHAudio);
+    CASE_RETURN_OTHER(kCodecMP2);
+    CASE_RETURN_OTHER(kCodecDTS);
   }
   return absl::nullopt;  // Not a 'default' to ensure compile-time checks.
 }
diff -up chromium-93.0.4577.42/net/base/mime_util.cc.27~ chromium-93.0.4577.42/net/base/mime_util.cc
--- chromium-93.0.4577.42/net/base/mime_util.cc.27~	2021-08-12 21:10:19.000000000 +0200
+++ chromium-93.0.4577.42/net/base/mime_util.cc	2021-08-15 05:51:57.697300770 +0200
@@ -167,6 +167,9 @@ static const MimeInfo kPrimaryMappings[]
     {"image/png", "png"},
     {"image/apng", "png"},
     {"image/webp", "webp"},
+    {"video/x-matroska", "mkv"},
+    {"audio/x-matroska", "mkv"},
+    {"video/x-msvideo", "avi"},
     {"multipart/related", "mht,mhtml"},
     {"text/css", "css"},
     {"text/html", "html,htm,shtml,shtm"},
@@ -686,6 +689,7 @@ static const char* const kStandardAudioT
   "audio/vorbis",
   "audio/wav",
   "audio/webm",
+  "audio/x-matroska",
   "audio/x-m4a",
   "audio/x-ms-wma",
   "audio/vnd.rn-realaudio",
@@ -708,8 +712,10 @@ static const char* const kStandardVideoT
   "video/webm",
   "video/x-dv",
   "video/x-m4v",
+  "video/x-matroska",
   "video/x-mpeg",
   "video/x-ms-asf",
+  "video/x-msvideo",
   "video/x-ms-wmv"
 };
 
diff -up chromium-93.0.4577.42/third_party/openscreen/src/cast/streaming/remoting.proto.27~ chromium-93.0.4577.42/third_party/openscreen/src/cast/streaming/remoting.proto
--- chromium-93.0.4577.42/third_party/openscreen/src/cast/streaming/remoting.proto.27~	2021-08-12 21:18:11.000000000 +0200
+++ chromium-93.0.4577.42/third_party/openscreen/src/cast/streaming/remoting.proto	2021-08-15 05:51:57.697300770 +0200
@@ -60,6 +60,8 @@ message AudioDecoderConfig {
     kCodecALAC = 15;
     kCodecAC3 = 16;
     kCodecMpegHAudio = 17;
+    kCodecMP2 = 18;
+    kCodecDTS = 19;
   }
 
   // Proto version of Chrome's media::SampleFormat.
@@ -451,4 +453,4 @@ message RpcMessage {
     // RPC_DS_READUNTIL_CALLBACK
     DemuxerStreamReadUntilCallback demuxerstream_readuntilcb_rpc = 401;
   };
-}
\ No newline at end of file
+}
