diff -up chromium-136.0.7103.59/third_party/node/README.chromium.omv~ chromium-136.0.7103.59/third_party/node/README.chromium
diff -up chromium-136.0.7103.59/third_party/node/update_node_binaries.omv~ chromium-136.0.7103.59/third_party/node/update_node_binaries
--- chromium-136.0.7103.59/third_party/node/update_node_binaries.omv~	2025-05-01 21:48:25.103075697 +0200
+++ chromium-136.0.7103.59/third_party/node/update_node_binaries	2025-05-01 21:48:29.415868982 +0200
@@ -19,7 +19,7 @@ set -eu
 cd "$(dirname "$0")"
 
 BASE_URL="https://nodejs.org/dist"
-NODE_VERSION="v22.11.0"
+NODE_VERSION="v23.11.0"
 
 upload=false  # Default value
 
