diff -up chromium-92.0.4515.131/base/test/BUILD.gn.7~ chromium-92.0.4515.131/base/test/BUILD.gn
--- chromium-92.0.4515.131/base/test/BUILD.gn.7~	2021-08-14 04:23:28.113072347 +0200
+++ chromium-92.0.4515.131/base/test/BUILD.gn	2021-08-14 04:37:15.215239352 +0200
@@ -198,9 +198,6 @@ static_library("test_support") {
     sources += [ "test_file_util_linux.cc" ]
     public_deps += [ ":fontconfig_util_linux" ]
     data_deps += [ "//third_party/test_fonts" ]
-    if (current_toolchain == host_toolchain) {
-      data_deps += [ ":do_generate_fontconfig_caches" ]
-    }
   }
 
   if (is_mac) {
