diff -up chromium-122.0.6261.69/build/config/compiler/BUILD.gn.32~ chromium-122.0.6261.69/build/config/compiler/BUILD.gn
--- chromium-122.0.6261.69/build/config/compiler/BUILD.gn.32~	2024-02-23 15:52:27.761799585 +0100
+++ chromium-122.0.6261.69/build/config/compiler/BUILD.gn	2024-02-23 15:53:05.838285897 +0100
@@ -805,7 +805,7 @@ config("compiler") {
 
       # We only use one version of LLVM within a build so there's no need to
       # upgrade debug info, which can be expensive since it runs the verifier.
-      ldflags += [ "-Wl,-mllvm,-disable-auto-upgrade-debug-info" ]
+#      ldflags += [ "-Wl,-mllvm,-disable-auto-upgrade-debug-info" ]
     }
 
     # TODO(https://crbug.com/1211155): investigate why this isn't effective on
diff -up chromium-122.0.6261.69/third_party/perfetto/gn/BUILD.gn.32~ chromium-122.0.6261.69/third_party/perfetto/gn/BUILD.gn
--- chromium-122.0.6261.69/third_party/perfetto/gn/BUILD.gn.32~	2024-02-23 15:52:27.509796342 +0100
+++ chromium-122.0.6261.69/third_party/perfetto/gn/BUILD.gn	2024-02-23 15:52:27.761799585 +0100
@@ -404,7 +404,6 @@ config("system_zlib_config") {
 if (enable_perfetto_zlib) {
   group("zlib") {
     if (perfetto_use_system_zlib) {
-      public_configs = [ "//gn:system_zlib_config" ]
     } else if (perfetto_root_path == "//") {
       public_configs = [ "//buildtools:zlib_config" ]
       public_deps = [ "//buildtools:zlib" ]
