diff -up chromium-114.0.5735.198/build/config/compiler/BUILD.gn.omv~ chromium-114.0.5735.198/build/config/compiler/BUILD.gn
--- chromium-114.0.5735.198/build/config/compiler/BUILD.gn.omv~	2023-07-13 06:34:15.975262606 +0200
+++ chromium-114.0.5735.198/build/config/compiler/BUILD.gn	2023-07-13 06:34:42.619561745 +0200
@@ -732,7 +732,8 @@ config("compiler") {
         # toolchain has this flag.
         # We only use one version of LLVM within a build so there's no need to
         # upgrade debug info, which can be expensive since it runs the verifier.
-        ldflags += [ "-Wl,-mllvm,-disable-auto-upgrade-debug-info" ]
+# clang16 doesn't have this flag either
+#        ldflags += [ "-Wl,-mllvm,-disable-auto-upgrade-debug-info" ]
       }
     }
 
