diff -up chromium-138.0.7204.92/gpu/config/gpu_finch_features.cc.63~ chromium-138.0.7204.92/gpu/config/gpu_finch_features.cc
--- chromium-138.0.7204.92/gpu/config/gpu_finch_features.cc.63~	2025-06-27 23:22:02.000000000 +0200
+++ chromium-138.0.7204.92/gpu/config/gpu_finch_features.cc	2025-07-03 03:12:08.442922756 +0200
@@ -229,7 +229,7 @@ BASE_FEATURE(kEnableDrDc,
 // Enable WebGPU on gpu service side only. This is used with origin trial and
 // enabled by default on supported platforms.
 #if BUILDFLAG(IS_APPLE) || BUILDFLAG(IS_WIN) || BUILDFLAG(IS_CHROMEOS) || \
-    BUILDFLAG(IS_ANDROID)
+    BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_LINUX)
 #define WEBGPU_ENABLED base::FEATURE_ENABLED_BY_DEFAULT
 #else
 #define WEBGPU_ENABLED base::FEATURE_DISABLED_BY_DEFAULT
diff -up chromium-138.0.7204.92/media/base/media_switches.cc.63~ chromium-138.0.7204.92/media/base/media_switches.cc
--- chromium-138.0.7204.92/media/base/media_switches.cc.63~	2025-07-03 03:11:03.333339486 +0200
+++ chromium-138.0.7204.92/media/base/media_switches.cc	2025-07-03 03:11:21.234731363 +0200
@@ -727,7 +727,7 @@ BASE_FEATURE(kUnifiedAutoplay,
 // on chromeos, but needs an experiment on linux.
 BASE_FEATURE(kAcceleratedVideoDecodeLinux,
              "AcceleratedVideoDecoder",
-#if BUILDFLAG(USE_VAAPI)
+#if 1 || BUILDFLAG(USE_VAAPI)
              base::FEATURE_ENABLED_BY_DEFAULT);
 #else
              base::FEATURE_DISABLED_BY_DEFAULT);
@@ -735,11 +735,11 @@ BASE_FEATURE(kAcceleratedVideoDecodeLinu
 
 BASE_FEATURE(kAcceleratedVideoDecodeLinuxGL,
              "AcceleratedVideoDecodeLinuxGL",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAcceleratedVideoEncodeLinux,
              "AcceleratedVideoEncoder",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Ignore the non-intel driver blacklist for VaapiVideoDecoder implementations.
 // Intended for manual usage only in order to gague the status of newer driver
