diff -up chromium-88.0.4324.150/chrome/browser/about_flags.cc.omv~ chromium-88.0.4324.150/chrome/browser/about_flags.cc
--- chromium-88.0.4324.150/chrome/browser/about_flags.cc.omv~	2021-02-12 07:36:21.538164728 +0100
+++ chromium-88.0.4324.150/chrome/browser/about_flags.cc	2021-02-12 07:38:44.558461489 +0100
@@ -2869,7 +2869,7 @@ const FeatureEntry kFeatureEntries[] = {
      FEATURE_VALUE_TYPE(ash::features::kSystemTrayMicGainSetting)},
 #endif  // OS_CHROMEOS
 
-#if defined(OS_LINUX) && !defined(OS_CHROMEOS) && !defined(OS_ANDROID)
+#if 0 && defined(OS_LINUX) && !defined(OS_CHROMEOS) && !defined(OS_ANDROID)
     {
         "enable-accelerated-video-decode",
         flag_descriptions::kAcceleratedVideoDecodeName,
