--- chromium-35.0.1916.153/build/common.gypi.p2~	2014-06-13 17:40:59.175459122 +0200
+++ chromium-35.0.1916.153/build/common.gypi	2014-06-13 17:41:33.952387294 +0200
@@ -806,7 +806,7 @@
           'use_allocator%': 'see_use_tcmalloc',
           # sysroot needs to be an absolute path otherwise it generates
           # incorrect results when passed to pkg-config
-          'sysroot%': '<!(cd <(DEPTH) && pwd -P)/arm-sysroot',
+          #'sysroot%': '<!(cd <(DEPTH) && pwd -P)/arm-sysroot',
         }], # OS=="linux" and target_arch=="arm" and chromeos==0
 
         ['OS=="linux" and branding=="Chrome" and buildtype=="Official" and chromeos==0', {
