diff -up chromium-136.0.7103.59/build/config/compiler/BUILD.gn.omv~ chromium-136.0.7103.59/build/config/compiler/BUILD.gn
--- chromium-136.0.7103.59/build/config/compiler/BUILD.gn.omv~	2025-05-01 21:38:46.468213794 +0200
+++ chromium-136.0.7103.59/build/config/compiler/BUILD.gn	2025-05-01 21:38:55.473130872 +0200
@@ -1640,31 +1640,8 @@ config("clang_revision") {
   }
 }
 
-# Controls the usage of a warning suppression mapping (WSM) file to suppress
-# warnings based on the path of the file they come from. It's controlled by the
-# `clang_warning_suppression_file` gn argument , which points to a text file
-# defining which warnings should be suppressed where.
-# See //build/config/warning_suppression.txt for an example file; this is the
-# file used by Chromium.
-#
-# To use a different file, or to use this functionality outside of chromium,
-# set the `clang_warning_suppression_file` argument to point to the new file,
-# e.g. by setting in the the project's .gn file.
 config("clang_warning_suppression") {
-  # Some build configs use older versions of clang that don't support WSMs
-  if (!is_nacl && default_toolchain != "//build/toolchain/cros:target" &&
-      !llvm_android_mainline && is_clang &&
-      clang_warning_suppression_file != "") {
-    from_build_root =
-        rebase_path(clang_warning_suppression_file, root_build_dir)
-    inputs = [ clang_warning_suppression_file ]
-    cflags = [
-      "-Xclang",
-      "--warning-suppression-mappings=" + from_build_root,
-    ]
-  }
 }
-
 config("rustc_revision") {
   if (rustc_revision != "") {
     # Similar to the above config, this is here so that all files get recompiled
