diff -up chromium-85.0.4164.2/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.h.20~ chromium-85.0.4164.2/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.h
--- chromium-85.0.4164.2/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.h.20~	2020-06-06 20:31:30.030944217 +0200
+++ chromium-85.0.4164.2/ui/base/cursor/ozone/bitmap_cursor_factory_ozone.h	2020-06-06 20:35:09.657735090 +0200
@@ -7,6 +7,7 @@
 
 #include <map>
 #include <vector>
+#include <algorithm>
 
 #include "base/component_export.h"
 #include "base/macros.h"
