diff -up chromium-79.0.3945.56/content/child/webthemeengine_impl_android.cc.gcc-name-clash chromium-79.0.3945.56/content/child/webthemeengine_impl_android.cc
--- chromium-79.0.3945.56/content/child/webthemeengine_impl_android.cc.gcc-name-clash	2019-11-26 16:52:30.000000000 -0500
+++ chromium-79.0.3945.56/content/child/webthemeengine_impl_android.cc	2019-12-03 12:48:21.210018632 -0500
@@ -228,7 +228,7 @@ void WebThemeEngineAndroid::Paint(
       native_theme_extra_params, NativeColorScheme(color_scheme));
 }
 
-blink::ForcedColors WebThemeEngineAndroid::ForcedColors() const {
+blink::ForcedColors WebThemeEngineAndroid::GetForcedColors() const {
   return forced_colors_;
 }
 
diff -up chromium-79.0.3945.56/content/child/webthemeengine_impl_android.h.gcc-name-clash chromium-79.0.3945.56/content/child/webthemeengine_impl_android.h
--- chromium-79.0.3945.56/content/child/webthemeengine_impl_android.h.gcc-name-clash	2019-12-03 12:48:21.213018564 -0500
+++ chromium-79.0.3945.56/content/child/webthemeengine_impl_android.h	2019-12-03 12:49:44.079153643 -0500
@@ -22,7 +22,7 @@ class WebThemeEngineAndroid : public bli
              const blink::WebRect& rect,
              const blink::WebThemeEngine::ExtraParams* extra_params,
              blink::WebColorScheme color_scheme) override;
-  blink::ForcedColors ForcedColors() const override;
+  blink::ForcedColors GetForcedColors() const override;
   void SetForcedColors(const blink::ForcedColors forced_colors) override;
 
  private:
diff -up chromium-79.0.3945.56/content/child/webthemeengine_impl_default.cc.gcc-name-clash chromium-79.0.3945.56/content/child/webthemeengine_impl_default.cc
--- chromium-79.0.3945.56/content/child/webthemeengine_impl_default.cc.gcc-name-clash	2019-11-26 16:52:30.000000000 -0500
+++ chromium-79.0.3945.56/content/child/webthemeengine_impl_default.cc	2019-12-03 12:48:21.215018520 -0500
@@ -232,7 +232,7 @@ void WebThemeEngineDefault::cacheScrollB
 }
 #endif
 
-blink::ForcedColors WebThemeEngineDefault::ForcedColors() const {
+blink::ForcedColors WebThemeEngineDefault::GetForcedColors() const {
   return ui::NativeTheme::GetInstanceForWeb()->UsesHighContrastColors()
              ? blink::ForcedColors::kActive
              : blink::ForcedColors::kNone;
diff -up chromium-79.0.3945.56/content/child/webthemeengine_impl_default.h.gcc-name-clash chromium-79.0.3945.56/content/child/webthemeengine_impl_default.h
--- chromium-79.0.3945.56/content/child/webthemeengine_impl_default.h.gcc-name-clash	2019-12-03 12:48:21.217018474 -0500
+++ chromium-79.0.3945.56/content/child/webthemeengine_impl_default.h	2019-12-03 12:50:30.808102006 -0500
@@ -39,7 +39,7 @@ class WebThemeEngineDefault : public bli
                                     int32_t vertical_arrow_bitmap_height,
                                     int32_t horizontal_arrow_bitmap_width);
 #endif
-  blink::ForcedColors ForcedColors() const override;
+  blink::ForcedColors GetForcedColors() const override;
   void SetForcedColors(const blink::ForcedColors forced_colors) override;
 };
 
diff -up chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.cc.gcc-name-clash chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.cc
--- chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.cc.gcc-name-clash	2019-11-26 16:52:30.000000000 -0500
+++ chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.cc	2019-12-03 12:48:21.220018407 -0500
@@ -6,7 +6,7 @@
 
 namespace content {
 
-blink::ForcedColors WebThemeEngineMac::ForcedColors() const {
+blink::ForcedColors WebThemeEngineMac::GetForcedColors() const {
   return forced_colors_;
 }
 
diff -up chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.h.gcc-name-clash chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.h
--- chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.h.gcc-name-clash	2019-12-03 12:48:21.223018339 -0500
+++ chromium-79.0.3945.56/content/child/webthemeengine_impl_mac.h	2019-12-03 12:51:07.790269715 -0500
@@ -13,7 +13,7 @@ class WebThemeEngineMac : public blink::
  public:
   ~WebThemeEngineMac() override {}
 
-  blink::ForcedColors ForcedColors() const override;
+  blink::ForcedColors GetForcedColors() const override;
   void SetForcedColors(const blink::ForcedColors forced_colors) override;
 
  private:
diff -up chromium-79.0.3945.56/third_party/blink/public/platform/web_theme_engine.h.gcc-name-clash chromium-79.0.3945.56/third_party/blink/public/platform/web_theme_engine.h
--- chromium-79.0.3945.56/third_party/blink/public/platform/web_theme_engine.h.gcc-name-clash	2019-12-03 12:48:21.225018294 -0500
+++ chromium-79.0.3945.56/third_party/blink/public/platform/web_theme_engine.h	2019-12-03 12:51:54.547217442 -0500
@@ -226,7 +226,7 @@ class WebThemeEngine {
     return base::nullopt;
   }
 
-  virtual ForcedColors ForcedColors() const { return ForcedColors::kNone; }
+  virtual ForcedColors GetForcedColors() const { return ForcedColors::kNone; }
   virtual void SetForcedColors(const blink::ForcedColors forced_colors) {}
 };
 
diff -up chromium-79.0.3945.56/third_party/blink/renderer/core/css/media_values.cc.gcc-name-clash chromium-79.0.3945.56/third_party/blink/renderer/core/css/media_values.cc
--- chromium-79.0.3945.56/third_party/blink/renderer/core/css/media_values.cc.gcc-name-clash	2019-11-26 16:52:32.000000000 -0500
+++ chromium-79.0.3945.56/third_party/blink/renderer/core/css/media_values.cc	2019-12-03 12:48:21.226018272 -0500
@@ -221,7 +221,7 @@ bool MediaValues::CalculatePrefersReduce
 
 ForcedColors MediaValues::CalculateForcedColors() {
   if (Platform::Current() && Platform::Current()->ThemeEngine())
-    return Platform::Current()->ThemeEngine()->ForcedColors();
+    return Platform::Current()->ThemeEngine()->GetForcedColors();
   else
     return ForcedColors::kNone;
 }
diff -up chromium-79.0.3945.56/third_party/blink/renderer/core/dom/document.cc.gcc-name-clash chromium-79.0.3945.56/third_party/blink/renderer/core/dom/document.cc
--- chromium-79.0.3945.56/third_party/blink/renderer/core/dom/document.cc.gcc-name-clash	2019-11-26 16:52:32.000000000 -0500
+++ chromium-79.0.3945.56/third_party/blink/renderer/core/dom/document.cc	2019-12-03 12:48:21.237018024 -0500
@@ -8624,7 +8624,7 @@ void Document::ColorSchemeChanged() {
 bool Document::InForcedColorsMode() const {
   return RuntimeEnabledFeatures::ForcedColorsEnabled() && Platform::Current() &&
          Platform::Current()->ThemeEngine() &&
-         Platform::Current()->ThemeEngine()->ForcedColors() !=
+         Platform::Current()->ThemeEngine()->GetForcedColors() !=
              ForcedColors::kNone;
 }
 
