diff -up chromium-62.0.3202.62/services/resource_coordinator/resource_coordinator_service.cc.dde535 chromium-62.0.3202.62/services/resource_coordinator/resource_coordinator_service.cc
--- chromium-62.0.3202.62/services/resource_coordinator/resource_coordinator_service.cc.dde535	2017-10-18 10:04:33.237966223 -0400
+++ chromium-62.0.3202.62/services/resource_coordinator/resource_coordinator_service.cc	2017-10-18 10:05:00.442438694 -0400
@@ -18,7 +18,8 @@ std::unique_ptr<service_manager::Service
   auto resource_coordinator_service =
       base::MakeUnique<ResourceCoordinatorService>();
 
-  return resource_coordinator_service;
+  return std::unique_ptr<service_manager::Service>(
+      resource_coordinator_service.release());
 }
 
 ResourceCoordinatorService::ResourceCoordinatorService()
