diff -up chromium-54.0.2840.59/build/linux/unbundle/libjpeg.gn.gnsystem chromium-54.0.2840.59/build/linux/unbundle/libjpeg.gn
--- chromium-54.0.2840.59/build/linux/unbundle/libjpeg.gn.gnsystem	2016-10-13 16:06:00.464796742 -0400
+++ chromium-54.0.2840.59/build/linux/unbundle/libjpeg.gn	2016-10-13 16:06:08.976631824 -0400
@@ -16,6 +16,10 @@ source_set("libjpeg") {
   libs = [ "jpeg" ]
 }
 
+config("system_libjpeg") {
+  defines = [ "USE_SYSTEM_LIBJPEG=1" ]
+}
+
 source_set("simd") {
 }
 
diff -up chromium-54.0.2840.59/build/linux/unbundle/libusb.gn.gnsystem chromium-54.0.2840.59/build/linux/unbundle/libusb.gn
--- chromium-54.0.2840.59/build/linux/unbundle/libusb.gn.gnsystem	2016-10-13 15:47:48.147891525 -0400
+++ chromium-54.0.2840.59/build/linux/unbundle/libusb.gn	2016-10-13 15:47:48.147891525 -0400
@@ -0,0 +1,24 @@
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/linux/pkg_config.gni")
+import("//build/shim_headers.gni")
+
+pkg_config("system_libusb") {
+  packages = [ "libusb-1.0" ]
+}
+
+shim_headers("libusb_shim") {
+  root_path = "src/libusb"
+  headers = [
+    "libusb.h",
+  ]
+}
+
+source_set("libusb") {
+  deps = [
+    ":libusb_shim",
+  ]
+  public_configs = [ ":system_libusb" ]
+}
