diff -up chromium-141.0.7390.54/gpu/config/gpu_finch_features.cc.53~ chromium-141.0.7390.54/gpu/config/gpu_finch_features.cc
--- chromium-141.0.7390.54/gpu/config/gpu_finch_features.cc.53~	2025-09-29 19:05:47.000000000 +0200
+++ chromium-141.0.7390.54/gpu/config/gpu_finch_features.cc	2025-10-04 04:19:21.751099802 +0200
@@ -198,7 +198,7 @@ BASE_FEATURE(EnableDrDc,
 // Enable WebGPU on gpu service side only. This is used with origin trial and
 // enabled by default on supported platforms.
 #if BUILDFLAG(IS_APPLE) || BUILDFLAG(IS_WIN) || BUILDFLAG(IS_CHROMEOS) || \
-    BUILDFLAG(IS_ANDROID)
+    BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_LINUX)
 #define WEBGPU_ENABLED base::FEATURE_ENABLED_BY_DEFAULT
 #else
 #define WEBGPU_ENABLED base::FEATURE_DISABLED_BY_DEFAULT
diff -up chromium-141.0.7390.54/media/base/media_switches.cc.53~ chromium-141.0.7390.54/media/base/media_switches.cc
--- chromium-141.0.7390.54/media/base/media_switches.cc.53~	2025-10-04 04:19:03.010840798 +0200
+++ chromium-141.0.7390.54/media/base/media_switches.cc	2025-10-04 04:20:18.873902635 +0200
@@ -672,17 +672,17 @@ BASE_FEATURE(UnifiedAutoplay, base::FEAT
 // on chromeos, but needs an experiment on linux.
 BASE_FEATURE(kAcceleratedVideoDecodeLinux,
              "AcceleratedVideoDecoder",
-#if BUILDFLAG(USE_VAAPI)
+#if 1 || BUILDFLAG(USE_VAAPI)
              base::FEATURE_ENABLED_BY_DEFAULT);
 #else
              base::FEATURE_DISABLED_BY_DEFAULT);
 #endif
 
-BASE_FEATURE(AcceleratedVideoDecodeLinuxGL, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(AcceleratedVideoDecodeLinuxGL, base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAcceleratedVideoEncodeLinux,
              "AcceleratedVideoEncoder",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Ignore the non-intel driver blacklist for VaapiVideoDecoder implementations.
 // Intended for manual usage only in order to gague the status of newer driver
