diff -up chromium-139.0.7258.127/skia/BUILD.gn.46~ chromium-139.0.7258.127/skia/BUILD.gn
--- chromium-139.0.7258.127/skia/BUILD.gn.46~	2025-08-18 03:08:15.164922563 +0200
+++ chromium-139.0.7258.127/skia/BUILD.gn	2025-08-18 03:31:07.346339379 +0200
@@ -191,7 +191,7 @@ config("skia_library_config") {
   if (use_blink && !use_system_freetype) {
     defines += [ "SK_FREETYPE_MINIMUM_RUNTIME_VERSION_IS_BUILD_VERSION" ]
   }
-  if (use_blink && enable_freetype) {
+  if (use_blink) {
     defines += [
       "SK_TYPEFACE_FACTORY_FREETYPE",
       "SK_FONTMGR_FREETYPE_EMPTY_AVAILABLE",
@@ -423,7 +423,7 @@ component("skia") {
   # default compile-in Fontations as well.
   if (use_blink) {
     # See SK_TYPEFACE_FACTORY_FREETYPE and SK_TYPEFACE_FACTORY_FONTATIONS
-    if (enable_freetype) {
+    if (true) {
       sources += skia_ports_freetype_sources
       sources += skia_ports_fontmgr_custom_sources
       sources += skia_ports_fontmgr_empty_sources
diff -up chromium-139.0.7258.127/third_party/BUILD.gn.46~ chromium-139.0.7258.127/third_party/BUILD.gn
--- chromium-139.0.7258.127/third_party/BUILD.gn.46~	2025-08-12 00:26:04.000000000 +0200
+++ chromium-139.0.7258.127/third_party/BUILD.gn	2025-08-18 03:08:16.876283601 +0200
@@ -55,7 +55,7 @@ group("jpeg_includes") {
 component("freetype_harfbuzz") {
   public_configs = []
   public_deps = []
-  if (enable_freetype) {
+  if (true) {
     if (use_system_freetype) {
       public_configs += [ "//build/linux:freetype_from_pkgconfig" ]
     } else {
@@ -73,6 +73,6 @@ buildflag_header("freetype_buildflags")
   header = "freetype_buildflags.h"
   flags = [
     "USE_SYSTEM_FREETYPE=$use_system_freetype",
-    "ENABLE_FREETYPE=$enable_freetype",
+    "ENABLE_FREETYPE=1",
   ]
 }
