--- ./services/tracing/perfetto/track_event_json_exporter.cc.orig	2019-06-07 22:56:31.947798420 +0100
+++ ./services/tracing/perfetto/track_event_json_exporter.cc	2019-06-08 13:02:42.003222377 +0100
@@ -530,6 +530,6 @@
   }
   // Even if |flags==0|, we need to call AddFlags to output instant event scope.
   builder.AddFlags(flags, id, event.id_scope());
-  return builder;
+  return base::Optional<JSONTraceExporter::ScopedJSONTraceEventAppender>(std::move(builder));
 }
 }  // namespace tracing
