diff -up chromium-128.0.6613.84/third_party/pdfium/BUILD.gn.25~ chromium-128.0.6613.84/third_party/pdfium/BUILD.gn
--- chromium-128.0.6613.84/third_party/pdfium/BUILD.gn.25~	2024-08-22 00:48:21.604075000 +0200
+++ chromium-128.0.6613.84/third_party/pdfium/BUILD.gn	2024-08-23 15:30:57.278640215 +0200
@@ -4,6 +4,7 @@
 
 import("//build/config/clang/clang.gni")
 import("//build/config/gclient_args.gni")
+import("//build/config/linux/pkg_config.gni")
 import("//testing/test.gni")
 import("pdfium.gni")
 
@@ -24,6 +25,13 @@ group("freetype_common") {
   }
 }
 
+if (use_system_libopenjpeg2) {
+  pkg_config("libopenjpeg2_from_pkgconfig") {
+    defines = [ "USE_SYSTEM_LIBOPENJPEG2" ]
+    packages = [ "libopenjp2" ]
+  }
+}
+
 config("pdfium_common_config") {
   cflags = []
   cflags_cc = []
@@ -31,7 +39,9 @@ config("pdfium_common_config") {
   include_dirs = [ "." ]
   defines = []
 
-  if (!use_system_libopenjpeg2) {
+  if (use_system_libopenjpeg2) {
+    configs = [ ":libopenjpeg2_from_pkgconfig" ]
+  } else {
     defines += [ "OPJ_STATIC" ]
   }
 
