--- chromium-75.0.3770.80/build/config/compiler/BUILD.gn.0503~	2019-06-06 20:50:40.560385017 +0100
+++ chromium-75.0.3770.80/build/config/compiler/BUILD.gn	2019-06-06 20:52:39.417377417 +0100
@@ -471,20 +471,6 @@
     }
   }
 
-  if (is_clang && !is_nacl && !use_xcode_clang) {
-    cflags += [ "-fcrash-diagnostics-dir=" +
-                rebase_path("//tools/clang/crashreports", root_build_dir) ]
-
-    cflags += [
-      # TODO(hans): Remove this once Clang generates better optimized debug info
-      # by default. https://crbug.com/765793
-      "-Xclang",
-      "-mllvm",
-      "-Xclang",
-      "-instcombine-lower-dbg-declare=0",
-    ]
-  }
-
   # C11/C++11 compiler flags setup.
   # ---------------------------
   if (is_linux || is_android || (is_nacl && is_clang) || current_os == "aix") {
